<h2>Lesson 14.2: Phylogenetic Classification</h2><h3>Lesson Objectives</h3><ul><li>Understand the concept of phylogenetic classification. </li><li>Outline how cladistics generates cladograms and identifies clades. </li><li>Compare phylogenetic and Linnaean classification systems. </li><li>Explain how nucleic acid base sequences are used in phylogenetic classification. </li></ul><h3>Introduction</h3><p>In the century after Linnaeus published his system of classification, ideas about classifying organisms began to change. In 1859, Darwin published his major work on evolution, <em>On the Origin of Species by Natural Selection.</em> After that, there was more and more interest in classifying organisms, incorporating the evolutionary history, including the genetic relationships, of the organisms.</p><h3>Phylogeny</h3><p>The evolutionary history of a group of genetically related organisms is called a <strong role="strong">phylogeny</strong>. It includes ancestor species and descendant species. A phylogeny is usually represented by a tree diagram called a <strong role="strong">phylogenetic tree</strong>. An early example of a phylogenetic tree is Darwin&rsquo;s &ldquo;Tree of Life&rdquo; (see <strong role="strong">Figure</strong>  1.7 <a href="#BioII-14-0102-01-fig" id="BioII-14-0102-01-text">below</a>). In this diagram, Darwin was trying to show how he thought evolution had occurred. The tree shows how species evolved through time, from the bottom of the tree to the top. As species evolved, they formed new branches on the tree of life. Some of these species eventually branched into additional descendant species. Others died out, or went extinct, without leaving any descendants.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2850" width="256px" /></div> <p><a href="#BioII-14-0102-01-text" id="BioII-14-0102-01-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.7 <caption>. This branching diagram represents the evolutionary histories of different species. It is the only diagram that originally appeared in Darwins famous 1859 book, </caption></p></div><p> </p><p>Modern biologists still use phylogenetic trees to represent evolutionary histories. A simple phylogenetic tree is shown in <strong role="strong">Figure</strong>  1.8 <a href="#BioII-14-0102-02-fig" id="BioII-14-0102-02-text">below</a>. The tips of the branches represent genetically related species. The branching points represent common ancestors. A <strong role="strong">common ancestor</strong> is the last ancestor species that two descendant species shared before they took different evolutionary paths. In the tree in <strong role="strong">Figure</strong>  1.8 <a href="#BioII-14-0102-02-fig" id="BioII-14-0102-02-text">below</a>, species 1 and 2 shared a more recent common ancestor with each other than with species 3. Therefore, species 1 and 2 are more closely related to one another than to species 3.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2839" width="256px" /></div> <p><a href="#BioII-14-0102-02-text" id="BioII-14-0102-02-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.8 <caption>. This phylogenetic tree shows how hypothetical species 1, 2, and 3 are related to one another through common ancestors.</caption></p></div><p> </p><p>Ancestor species are like your own ancestors. Your most recent common ancestor with any siblings you may have is a shared parent. Your most recent common ancestor with a first cousin is a shared grandparent. Your most recent common ancestor with a second cousin is a shared great-grandparent. In general, the more distant the relationship between you and relatives in your own generation, the farther in the past you shared a common ancestor. The same holds true for related species. The more distant the relationship between two related species, the farther back in time they shared a common ancestor.</p><p>The most common method of incorporating information into phylogenetic trees is called <strong role="strong">cladistics</strong>. Cladistics depicts hypotheses about how organisms are related, based on traits of ancestor and descendent species. Cladistics was developed in the 1950s by a scientist named Willi Hennig. Over the next several decades, it became very popular, and is still widely used today.</p><p>The term <em>cladistics</em> comes from the word <em>clade</em>. A <strong role="strong">clade</strong> is a group of organisms that includes an ancestor species and all of its descendants. A diagram showing evolutionary relationships within one or more clades is called a <strong role="strong">cladogram</strong>. A clade is a relative concept. How you define a clade depends on which species you are interested in classifying. Small clades can includes as few as two species and their common ancestor. The larger clades can include many more species and their common ancestors.</p><p>As another example, consider the cladogram of insect phylogeny shown in <strong role="strong">Figure</strong>  1.9 <a href="#BioII_14_0102_04-fig" id="BioII_14_0102_04-text">below</a>. According to this cladogram, beetles first branched off from their common ancestor with other insects. Then, the group that includes wasps, bees, and ants branched off. Finally, flies branched off from their common ancestor with butterflies and moths. All insects can be considered a clade because they have a common ancestor. Butterflies, moths, and flies can also be considered a clade for the same reason. Can you identify other clades in <strong role="strong">Figure</strong>  1.9 <a href="#BioII_14_0102_04-fig" id="BioII_14_0102_04-text">below</a>? For example, can you find the clade of all nonbeetle insects?</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2841" width="256px" /></div> <p><a href="#BioII_14_0102_04-text" id="BioII_14_0102_04-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.9 <caption>Cladogram of Insect Phylogeny. Based on this cladogram, flies shared a more recent common ancestor with butterflies and moths than either group shared with other insects. What other evolutionary relationships does the cladogram reveal?</caption></p></div><p> </p><h4>Generating Cladograms</h4><p>How do scientists construct cladograms like the one in <strong role="strong">Figure</strong>  1.10 <a href="#BioII-14-0102-05-fig" id="BioII-14-0102-05-text">below</a>? The starting point is a set of data on traits of a group of related species. The traits could be physical traits, genetic traits, or both (see <strong role="strong">Evidence for Evolutionary Relationships</strong> below). The next step is deciding which traits were inherited from the common ancestor and which traits evolved only in a descendant species after splitting off from the common ancestor. Traits inherited from a common ancestor are called <strong role="strong">ancestral traits</strong>. Traits that evolved since two groups shared a common ancestor are called <strong role="strong">derived traits</strong> and both types of traits are illustrated in <strong role="strong">Figure</strong>  1.10 <a href="#BioII-14-0102-05-fig" id="BioII-14-0102-05-text">below</a>.</p><p><strong role="strong">Ancestral and Derived Traits in Cladistic Analysis</strong></p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2844" width="256px" /></div> <p><a href="#BioII-14-0102-05-text" id="BioII-14-0102-05-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.10 <caption>In this cladogram, the ancestor species has traits A, B, and C, so these are ancestral traits. During the process of evolution, trait A evolves to trait a and trait B evolves to trait b. These new traits (a and b) are derived traits. Organisms can be classified into separate groups (species #1 or species #2) on the basis of these derived traits.</caption></p></div><p> </p><p>Consider birds as an example. A derived trait in birds is feathers. The trait is present only in birds and was not inherited from a common ancestor of birds and other organisms. An example of an ancestral trait in birds is the presence of eyes. Eyes are present not only in birds but also in many other groups of animals that share a common ancestor with birds. Therefore, the presence of feathers can identify an organism as a bird, but the presence of eyes cannot. In cladistics, the sharing of derived traits is the most important evidence for evolutionary relationships. Organisms with the same derived traits (such as feathers) are grouped in the same clade.</p><blockquote><strong role="strong">A derived trait is not necessarily an entirely new trait. More often it is a modified form of an ancestral trait. For example, birds evolved feathers from the scales that were already present in their reptile ancestor. Similarly, mammals evolved fur from the scales of their reptile ancestor.</strong></blockquote><p>More than one possible cladogram usually can be created from the same set of data. In fact, the number of possible cladograms increases exponentially with the number of species included in the analysis. Only one cladogram is possible with two species. More than 100 cladograms are possible with five species. With nine species, more than two million cladograms are possible! <strong role="strong">Figure</strong>  1.11 <a href="#six_cladograms-fig" id="six_cladograms-text">below</a> shows just six of the many possible cladograms that can be generated for five species.</p><p><strong role="strong">Six Possible Cladograms for Five Hypothetical Species</strong></p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2855" width="256px" /></div> <p><a href="#six_cladograms-text" id="six_cladograms-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.11 <caption>The same set of data on five related species may generate over 100 possible cladograms. Just six are shown here. In cladogram 1 (top, left), D and E share a more recent common ancestor than either shares with C. In cladogram 2 (top, middle), C and D share a more recent common ancestor than either shares with E. Compare how each of the remaining cladograms differs from the others.</caption></p></div><p> </p><p>How do scientists know which of many possible cladograms is the &ldquo;right&rdquo; one? There is no right or wrong cladogram. However, some cladograms fit the facts better than others. Statistical methods can be used to determine which cladogram best fits a particular data set. An important deciding factor is parsimony. <strong role="strong">Parsimony</strong> means choosing the simplest explanation from among all possible explanations. In cladistics, parsimony usually means choosing the cladogram with the fewest branching points.</p><p>A cladogram shows just one of many possible phylogenies for a group of organisms. It can provide insights about how evolution occurred. However, a cladogram should not be considered a model of actual evolutionary events. It does not necessarily show what really happened. It just shows what could have happened.</p><h3>Phylogenetic Classification</h3><p>A cladogram shows how species may be related by descent from a common ancestor. A classification of organisms on the basis of such relationships is called a <strong role="strong">phylogenetic classification</strong>. A phylogenetic classification involves placing organisms in a clade with their common ancestor. Consider the cladogram in <strong role="strong">Figure</strong>  1.12 <a href="#BioII-14-0102-07-fig" id="BioII-14-0102-07-text">below</a>. It groups birds in the same clade as reptiles, because a variety of evidence suggests that birds evolved from a reptile ancestor. The cladogram places mammals in a separate clade, because evidence suggests that mammals evolved from a different ancestor.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2842" width="256px" /></div> <p><a href="#BioII-14-0102-07-text" id="BioII-14-0102-07-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.12 <caption>. This cladogram represents the evolutionary history of reptiles, birds, and mammals. The reptile clade includes birds. Mammals are in a separate clade.</caption></p></div><p> </p><p><strong role="strong">Figure</strong>  1.13 <a href="#BioII-14-0102-08-fig" id="BioII-14-0102-08-text">below</a> shows the phylogenetic classification of reptiles, birds, and mammals based on the cladogram in <strong role="strong">Figure</strong>  1.12 <a href="#BioII-14-0102-07-fig" id="BioII-14-0102-07-text">below</a>. Birds are grouped with reptiles in one clade, called the Sauropsids. Mammals and their reptile-like ancestor are grouped in a separate clade, called the Synapsids. Compare this phylogenetic classification with the Linnaean classification, also shown in <strong role="strong">Figure</strong>  1.13 <a href="#BioII-14-0102-08-fig" id="BioII-14-0102-08-text">below</a>. In the Linnaean classification, reptiles, birds, and mammals are all placed in separate classes based on differences in physical traits. This classification artificially separates both birds and mammals from their reptilian ancestors. It also illustrates the difficulty of showing evolutionary relationships with Linnaean taxonomy.</p><h4>Phylogenetic and Linnaean Classifications of Reptiles, Birds, and Mammals</h4><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2847" width="256px" /></div> <p><a href="#BioII-14-0102-08-text" id="BioII-14-0102-08-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.13 <caption>The cladistic classification on the right assumes that mammals and birds evolved from different reptile ancestors. Mammals are placed in one clade, and birds are placed in another clade (with modern reptiles). Compare this classification with the Linnaean classification on the left. Why are birds and reptiles placed in separate classes in the Linnean taxonomy</caption></p></div><p> </p><p>Both phylogenetic and Linnaean classification systems have advantages and drawbacks (see the point by point comparison in the two lists, below). As an overall approach, most biologists think that phylogenetic classification is preferable to Linnaean classification. This is because it is based on evolutionary relationships and not just similarities in physical traits that may or may not have evolutionary significance. However, both approaches have a place in the classification of organisms. Linnaean binomial names are still needed to identify species, because phylogenetics does not include a method for naming species. In addition, many higher taxa in the Linnaean system, such as birds and mammals, remain useful in phylogenetic classifications. This is because they are also clades.</p><h4>Phylogenetic Classification</h4><ol start><li>Treats all levels of a cladogram as equivalent. </li><li>Places no limit on the number of levels in a ladogram. </li><li>Primary goal is to show the process of evolution. </li><li>It is limited to organisms that are related by ancestry. </li><li>Does not include a method for naming species. </li></ol><h4>Linnaean Classification</h4><ol start><li>Treats each taxa uniquely and has a special name or each (e.g., genus, species). </li><li>Has fixed numbers and types of taxa. </li><li>Primary goal is to group species based on similarities in physical traits. </li><li>Can include any organisms without regard to ancestry. </li><li>Has a method for giving unique names to species. </li></ol><blockquote><strong role="strong">Phenetics is an older method to classify organisms. Phenetics is based on overall similarity, usually in morphology or other observable traits, regardless of their evolutionary relation. Phenetics has largely been replaced by cladistics for research into evolutionary relationships among species. Phenetic techniques include various forms of clustering and ordination of traits. These are sophisticated ways of reducing the variation displayed by organisms to a manageable level. Phenetic analyses do not distinguish between traits that are inherited from an ancestor  and traits that evolved anew in one or several lineages. Consequently, phenetic analyses can be misled by convergent evolution and adaptive radiation.</strong></blockquote><h3>Evidence for Evolutionary Relationships</h3><p>Traditionally, evidence for evolutionary relationships included similarities in physical traits of form or function. For example, in Linnaean taxonomy, homeothermy (warm-bloodedness) is one of the traits used to separate both birds and mammals from other animals (see <strong role="strong">Figure</strong>  1.13 <a href="#BioII-14-0102-08-fig" id="BioII-14-0102-08-text">above</a>). However, this trait is not suitable for showing evolutionary relationships between birds and mammals. This is because birds and mammals did not inherit the trait of homeothermy from a common ancestor. Both groups independently evolved the trait. The presence of homeothermy in both birds and mammals is an example of convergent evolution (see the <em>History of Life</em> chapter). In general, convergent evolution may make two groups seem to be more closely related than they really are. Using such traits for phylogenetic analysis can lead to misleading phylogenetic classifications.</p><p>Similarities among nucleic acid base sequences provide some of the most direct evidence of evolutionary relationships (see the <em>Evolutionary Theory</em> chapter). Nucleic acids directly control genetic traits and copies of nucleic acids are actually passed from parents to offspring. Therefore, similarities in these traits are likely to reflect shared ancestry. By the 1960s, scientists had found ways to sequence the bases in nucleic acids. This coincided with the growing popularity of cladistics. In cladistic analysis, similar nucleic acid base sequences are assumed to indicate descent from a common ancestor. The more similar the sequences, the more recently two groups are assumed to have shared a common ancestor.</p><p>Many base sequence comparisons have confirmed genetic relationships that were assumed on the basis of similarities in physical traits. For example, 96 percent of the DNA in humans and chimpanzees is the same. This agrees, in general, with the Linnaean classification of chimpanzees as close human relatives (see Lesson 14.1).</p><p>Most biologists interested in taxonomy now use nucleic acid sequences or other related molecular data to classify organisms. However, using nucleic acid base sequences for phylogenetic analysis is not without its drawbacks. Two of the drawbacks are:</p><ul><li>Data on nucleic acids can rarely be obtained for extinct species. This is true even for species represented by fossils. Fossil DNA and RNA generally are not sufficient in quantity or quality to be useful for such analyses. </li><li>Base sequence data may be influenced by horizontal gene transfer. This occurs when an organism passes DNA to an unrelated organism. First discovered in bacteria in 1959, it is now known to be common in bacteria and some other microorganisms. Horizontal gene transfer can make species seem more closely related than they really are. </li></ul><p>Because of horizontal gene transfer, some biologists have started to question whether phylogenetic trees are the best way to show evolutionary relationships. This is especially true for those biologists that are interested in classifying bacteria. An entirely new process of determining evolutionary relationships may be needed in order to include horizontal gene transfer.</p><h3>Lesson Summary</h3><ul><li>Phylogeny is the evolutionary history of a group of genetically related organisms. It is usually represented by a diagram called a phylogenetic tree. </li><li>Cladistics is the most widely used method of generating phylogenetic trees. It is based on evolutionary ancestry and generates trees called cladograms. Cladistics also identifies clades, which are groups of organisms that include an ancestor species and its descendants. </li><li>Classifying organisms on the basis of descent from a common ancestor is called phylogenetic classification. Phylogenetic classification may or may not agree with Linnaean taxonomy, which is based on similarities in physical traits regardless of ancestry. </li><li>The most direct evidence for evolutionary relationships is similarity in base sequences of the nucleic acids DNA and RNA. The more similar the base sequences of two species, the more closely related the species are assumed to be. </li></ul><h3>Review Questions</h3><ol start><li>What is a phylogeny?</li><li>Define cladistics.</li><li>What does phylogenetic classification involve?</li><li>Why are nucleic acid base sequences directly related to evolution?</li><li>In cladogram 6 of <strong role="strong">Figure</strong>  1.11 <a href="#six_cladograms-fig" id="six_cladograms-text">above</a>, explain how the five species are related to one.</li><li>Identify an ancestral trait and a derived trait in mammals. Explain your answer.</li><li>Explain why a cladogram represents only one hypothesis about how evolution occurred.</li><li>Compare the advantages of Linnaean and phylogenetic classification systems.</li></ol><h3>Further Reading / Supplemental Links</h3><ul><li><a href="http://evolution.berkeley.edu/evolibrary/article/0_0_0/phylogenetics_01"> </a><a href="http://evolution.berkeley.edu/evolibrary/article/0_0_0/phylogenetics_01">http://evolution.berkeley.edu/evolibrary/article/0_0_0/phylogenetics_01</a></li><li>D.Graham Burnett, Trying Leviathan: The Nineteenth-Century Court Case that Put the Whale on Trial and Challenged the Order of Nature. Princeton University Press, 2007. </li><li><a href="http://evolution.berkeley.edu/evolibrary/article/0_0_0/phylogenetics_01"> Jan Sapp (ed.) </a><a href="http://evolution.berkeley.edu/evolibrary/article/0_0_0/phylogenetics_01">http://evolution.berkeley.edu/evolibrary/article/0_0_0/phylogenetics_01</a> Jan Sapp (ed.)</li><li> Microbial Phylogeny and Evolution: Concepts and Controversies. Oxford University Press, 2005. </li><li><a href="http://evolution.berkeley.edu/evolibrary/article/0_0_0/phylogenetics_01"> N. R. Scott-Ram </a><a href="http://evolution.berkeley.edu/evolibrary/article/0_0_0/phylogenetics_01">http://evolution.berkeley.edu/evolibrary/article/0_0_0/phylogenetics_01</a> N. R. Scott-Ram</li><li>Transformed Cladistics, Taxonomy, and Evolution. Cambridge University Press, 2009. </li><li><a href="http://evolution.berkeley.edu/evolibrary/article/0_0_0/phylogenetics_01"> </a><a href="http://evolution.berkeley.edu/evolibrary/article/0_0_0/phylogenetics_01-08">http://evolution.berkeley.edu/evolibrary/article/0_0_0/phylogenetics_01-08</a></li><li><a href="http://mansfield.osu.edu/~sabedon/biol3005.htm"> </a><a href="http://mansfield.osu.edu/~sabedon/biol3005.htm">http://mansfield.osu.edu/~sabedon/biol3005.htm</a></li><li><a href="http://news.nationalgeographic.com/news/2005/08/0831_050831_chimp_genes.html"> </a><a href="http://news.nationalgeographic.com/news/2005/08/0831_050831_chimp_genes.html">http://news.nationalgeographic.com/news/2005/08/0831_050831_chimp_genes.html</a></li><li><a href="http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/T/Taxonomy.html"> </a><a href="http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/T/Taxonomy.html">http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/T/Taxonomy.html</a></li><li><a href="http://www.genome.gov/12514316"> </a><a href="http://www.genome.gov/12514316">http://www.genome.gov/12514316</a></li><li><a href="http://www.palaeos.com/Systematics/Cladistics/incompatable.html"> </a><a href="http://www.palaeos.com/Systematics/Cladistics/incompatable.html">http://www.palaeos.com/Systematics/Cladistics/incompatable.html</a></li></ul><h3>Vocabulary</h3><dl><dt>ancestral traits </dt><dd>Traits inherited from a common ancestor. </dd></dl><dl><dt>clade </dt><dd>Group of organisms that includes an ancestor species and all of its descendants. </dd></dl><dl><dt>cladistics </dt><dd>Method of making evolutionary trees based on comparisons of traits of ancestor and descendant species. </dd></dl><dl><dt>cladogram </dt><dd>Diagram showing evolutionary relationships within one or more clades. </dd></dl><dl><dt>common ancestor </dt><dd>Last ancestral species that two descendant species shared before they took different evolutionary paths. </dd></dl><dl><dt>derived traits </dt><dd>Traits that evolved since two groups shared a common ancestor. </dd></dl><dl><dt>parsimony </dt><dd>Choosing the simplest explanation from among all possible explanations. </dd></dl><dl><dt>phylogenetic classification </dt><dd>Classification of organisms on the basis of evolutionary relationships. </dd></dl><dl><dt>phylogenetic tree </dt><dd>Diagram representing a phylogeny. </dd></dl><dl><dt>phylogeny </dt><dd>Evolutionary history of a group of genetically related organisms. </dd></dl><h3>Points to Consider</h3><p>When Linnaeus developed his classification system in the early 1700s, he knew almost nothing about microorganisms (microscopic organisms). Therefore, he did not include microorganisms in his taxonomy.</p><ul><li>How do you think microorganisms should be classified? </li><li>Where do you think microorganisms should be placed in Linnaean taxonomy? </li><li>Do you think a new taxon might be needed for microorganisms? </li></ul>