<h2>Lesson 8.4: Regulation of Gene Expression</h2><h3>Lesson Objectives</h3><ul><li>Describe general mechanisms of gene expression. </li><li>Differentiate between a cis-regulatory element and a trans-acting factor. </li><li>Define a transcription factor. </li><li>Define an operon. </li><li>Describe how the lac operon regulates transcription. </li><li>Describe the role of the TATA box. </li><li>Express the importance of gene regulation during development. </li><li>Describe the role of homeobox genes and gap genes. </li><li>Discuss gene regulation in terms of the development of cancer. </li></ul><h3>Introduction</h3><p>Each of your cells has about 22,000 genes. In fact, all of your cells have the same genes. So do all of your cells make the same proteins? Do all 22,000 genes get turned into proteins in every cell? Of course not. If they did, then all your cells would do the same thing. You have cells with different functions because you have cells with different proteins. And your cells have different proteins because they &ldquo;use&rdquo; different genes. The regulation of gene expression, or gene regulation, includes the mechanism to turn genes &ldquo;on&rdquo; and transcribe the gene into RNA. Any aspect of a gene's expression may be regulated, from the onset of transcription to the post-translational modification of a protein. It is this regulation that determines when and how much of a protein to make, giving a cell its specific structure and function.</p><h3>Mechanisms of Regulation</h3><p>Any step of gene expression may be modulated, from the DNA-RNA transcription step to post-translational modification of a protein. Following is a list of stages where gene expression is regulated:</p><ul><li>Chemical and structural modification of DNA or chromatin </li><li>Transcription </li><li>Translation </li><li>Post-transcriptional modification </li><li>RNA transport </li><li>mRNA degradation </li><li>Post-translational modifications </li></ul><p>In this lesson, we will focus on regulation at the level of transcription. We have previously discussed that during transcription RNA polymerase reads the DNA template to make a complementary strand of RNA. The genes to which RNA polymerase binds is a highly regulated process. When RNA polymerase binds to a gene, it binds to the <strong role="strong">promoter</strong>, a segment of DNA that allows a gene to be transcribed. The promoter helps RNA polymerase find the start of a gene.</p><p>Gene regulation at the level of transcription controls when transcription occurs as well as how much RNA is created. This regulation is controlled by <strong role="strong">cis-regulatory elements</strong> and <strong role="strong">trans-acting factors</strong>. A cis-regulatory element is a region of DNA which regulates the expression of a gene or multiple genes located on that same strand of DNA. These cis-regulatory elements are often the binding sites of one or more trans-acting factors, usually a regulatory protein which interacts with RNA polymerase. A cis-regulatory element may be located in a gene&rsquo;s promoter region, in an intron, or in the 3' region.</p><p>A <strong role="strong">regulatory protein</strong>, or a <strong role="strong">transcription factor</strong>, is a protein involved in regulating gene expression. It is usually bound to a cis-regulatory element. Regulatory proteins often must be bound to a cis-regulatory element to switch a gene on (activator), or to turn a gene off (repressor).</p><p>Transcription of a gene by RNA polymerase can be regulated by at least five mechanisms:</p><ul><li><strong role="strong">Specificity factors</strong> (proteins) alter the specificity of RNA polymerase for a promoter or set of promoters, making it more or less likely to bind to the promoter and begin transcription. </li><li><strong role="strong">Repressors</strong> (proteins) bind to non-coding sequences on the DNA that are close to or overlap the promoter region, impeding RNA polymerase's progress along the strand. </li><li><strong role="strong">Basal factors</strong>, transcription factors that help position RNA polymerase at the start of a gene. </li><li>Enhancers are sites on the DNA strand that are bound by activators in order to loop the DNA, bringing a specific promoter to the initiation complex. An <strong role="strong">initiation complex</strong> is composed of RNA polymerase and trans-acting factors. </li><li><strong role="strong">Activators</strong> (proteins) that enhance the interaction between RNA polymerase and a particular promoter.</li></ul><p>As the organism grows more sophisticated, gene regulation becomes more complex, though prokaryotic organisms possess some highly regulated systems. Some human genes are controlled by many activators and repressors working together. Obviously, a mutation in a cis-regulatory region, such as the promoter, can greatly affect the proper expression of a gene. It may keep the gene permanently off, such that no protein can be made, or it can keep the gene permanently on, such that the corresponding protein is constantly made. Both of these can have detremental effects on the cell.</p><h4>Prokaryotic Gene Regulation</h4><p>In prokaryotes, a combination of activators and repressors determines whether a gene is transcribed. As you know, prokaryotic organisms are fairly simple organisms with much less DNA. Prokaryotic genes are arranged in <strong role="strong">operons</strong>, a region of DNA with a promoter, an operator (defined below), and one or more genes that encode proteins needed to perform a certain task. To maintain homeostasis (and survive), the organism must quickly adapt changing environmental conditions. The regulation of transcription plays a key role in this process.</p><p>For a bacteria, many aspects of gene regulation are due to the presence or absence of certain nutrients. In prokaryotes, repressors bind to regions called <strong role="strong">operators</strong> that are generally located immediately downstream from the promoter. Activators bind to the upstream portion of the promoter.</p><h4>The Lac Operon</h4><p>The <strong role="strong">lac operon</strong> (<strong role="strong">Figure</strong>  1.22 <a href="#Bio_Lac_operon-fig" id="Bio_Lac_operon-text">below</a>) is an operon required for the transport and metabolism of lactose in <em>E. coli</em>. The lac operon is regulated by the availability of lactose. The lac operon consists of a promoter, an operator, three adjacent structural genes which code for enzymes and a terminator. The three genes are: lacZ, lacY, and lacA. All three genes are controlled by the same regulatory elements.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2678" width="256px" /></div> <p><a href="#Bio_Lac_operon-text" id="Bio_Lac_operon-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.22 <caption>The lac operon. The lac operon contains genes for three enzymes, lac, lacY, and lac A, as well as the promoter, operator, and terminatory regulatory regions.</caption></p></div><p> </p><p>In bacteria, the lac repressor protein blocks the synthesis of enzymes that digest lactose when there is no lactose present (<strong role="strong">Figure</strong>  1.23 <a href="#Bio8-4-2-fig" id="Bio8-4-2-text">below</a>). When lactose is present, it binds to the repressor, causing it to detach from the DNA strand.</p><p>Specific control of the lac operon depends on the availability of lactose. The enzymes needed to metabolize lactose are not produced when lactose is not present. When lactose is available, and therefore needs to be metabolized, the operon is turned on, RNA polymerase binds to the promoter, and the three genes are transcribed into a single mRNA molecule. However, if lactose is not present (and therefore does not need to be metabolized), the operon is turned off by the lac repressor protein (<strong role="strong">Figure</strong>  1.23 <a href="#Bio8-4-2-fig" id="Bio8-4-2-text">below</a>).</p><p>The lacI gene, which encodes the lac repressor, lies near the lac operon and is always expressed (constitutive). Therefore, the lac repressor protein is always present in the bacteria. In the absence of lactose, the lac repressor protein will bind to the operator, just past the promoter in the lac operon. The repressor blocks the binding of RNA polymerase to the promoter, keeping the operon turned off (<strong role="strong">Figure</strong>  1.23 <a href="#Bio8-4-2-fig" id="Bio8-4-2-text">below</a>).</p><p>When lactose is available, a lactose metabolite called allolactose binds to the repressor. This interaction causes a conformational change in the repressor shape and the repressor falls off the operator, allowing RNA polymerase to bind to the promoter and initiate transcription (<strong role="strong">Figure</strong>  1.23 <a href="#Bio8-4-2-fig" id="Bio8-4-2-text">below</a>).</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2667" width="256px" /></div> <p><a href="#Bio8-4-2-text" id="Bio8-4-2-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.23 <caption>Regulation of the lac operon. When lactose is present,  (red) binds to the promoter (P) and the three genes are expressed, producing a single mRNA for the three genes. When lactose is unavailable, the lac repressor (yellow) binds to the operator (O) and inhibits the binding of RNA polymerase to the promoter. The three genes are not expressed. For an animation of the Lac Operon, see </caption></p></div><p> </p><h4>Eukaryotic Gene Regulation</h4><p>As previously discussed, all your cells have the same DNA (and therefore the same genes), yet they have different proteins because they express different genes. In eukaryotic cells, the start of transcription is one of the most complex aspects of gene regulation. Transcriptional regulation involves the formation of an initiation complex involving interactions between a number of transcription factors, <em>cis</em>-regulatory elements, and <strong role="strong">enhancers</strong>, distant regions of DNA that can loop back to interact with a gene&rsquo;s promoter. These regulatory elements occur in unique combinations within a given cell type, resulting in only necessary genes being transcribed in certain cells. Transcription factors bind to a DNA strand, allowing RNA polymerase to bind and start transcription.</p><p>Each gene has unique cis-regulatory sequences, only allowing specific transcription factors to bind. However, there are common regulatory sequences found in most genes. The <strong role="strong">TATA box</strong> is a <em>cis</em>-regulatory element found in the promoter of most eukaryotic genes. It has the DNA sequence 5'-TATAAA-3' or a slight variant, and has been highly conserved throughout evolution. When the appropriate cellular signals are present, RNA polymerase binds to the TATA box, completing the initiation complex. A number of transcription factors first bind to the TATA box while other transcription factors bind to the previously attached factors, forming a multi-protein complex. It is only when all the appropriate factors are bound that RNA polymerase will recognize the complex and bind to the DNA, initiating transcription.</p><p>One of the more complex eukaryotic gene regulation processes is during development. What genes must be turned on during development so that tissues and organs form from simple cells?</p><h3>Regulation of Gene Expression During Development</h3><p>What makes the heart form during development? What makes the skin form? What makes a structure become an arm instead of a leg? These processes occur during development because of a highly specific pattern of gene expression. This intensely regulated pattern of gene expression turns genes on in the right cell at the right time, such that the resulting proteins can perform their necessary functions to ensure proper development. Transcription factors play an extremely important role during development. Many of these proteins can be considered master regulatory proteins, in the sense that they either activate or deactivate the transcription of other genes and, in turn, these secondary gene products can regulate the expression of still other genes in a regulatory cascade. Homeobox and gap genes are important transcription factors during development.</p><h4>Homeobox Genes</h4><p><strong role="strong">Homeobox genes</strong> contain a highly conserved DNA sequence known as a homeobox and are involved in the regulation of genes important to development. A homeobox is about 180 base pairs long; it encodes a 60 amino acid domain within the protein (known as the homeodomain), which can bind DNA. Proteins with a homeodomain are therefore transcription factors. These factors typically switch on series of other genes, for instance, the genes needed to encode the proteins to make a leg.</p><p>A particular subgroup of homeobox genes are the <strong role="strong">Hox genes</strong>. Protein products of Hox genes function in patterning the body, providing the placement of certain body parts during development. In other words, Hox genes determine where limbs and other body segments will grow in a developing fetus or larva. Mutations in any one of these genes can lead to the growth of extra, typically non-functional body parts in invertebrates. The Antennapedia mutation in <em>Drosophila</em> results in a leg growing from the head in place of an antenna. A mutation in a vertebrate Hox genes usually results in miscarriage.</p><h4>Gap Genes</h4><p>A <strong role="strong">gap gene</strong> controls the shape of a developing zygote early in its development. The products of these genes produce gaps in a rather uniform arrangement of cells (<strong role="strong">Figure</strong>  1.24 <a href="#Bio8-4-3-fig" id="Bio8-4-3-text">below</a>). One example of this is the Kruppel gene, which regulates the activity of a number of other genes. Gap genes encode transcription factors, and the Kruppel gene is a zinc-finger protein. A <strong role="strong">zinc finger</strong> is a DNA binding region within the protein. A zinc finger consists of two antiparallel &beta; sheets and an &alpha; helix with a zinc ion, which is important for the stability of this region. Gap genes control the expression of other genes within specific regions of cells in the developing organism. This allows specific genes to be expressed in certain cells at the appropriate stage of development.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2676" width="256px" /></div> <p><a href="#Bio8-4-3-text" id="Bio8-4-3-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.24 <caption>Gap gene expression. Shown is the expression pattern of four gap genes, Kruppel, Giant, Knirps, and Tailless, in a developing  embryo. Note how the expression of these genes creates an unique pattern resulting in gaps in what was a rather uniform arrangement of cells.</caption></p></div><p> </p><h3>Regulation of Gene Expression in Cancer</h3><p>As discussed in the last lesson, carcinogenesis depends on both the activation of oncogenes and deactivation of tumor suppressor genes. At least two separate mutations are necessary to develop cancer. For example, a mutation in a proto-oncogene would not necessarily lead to cancer, as normally functioning tumor suppressor genes would counteract the effects of the oncogene. It is the second mutation in the tumor suppressor gene that could lead to uncontrolled cell growth and possibly cancer. Both oncogenes and tumor suppressor genes play an important role in gene regulation and cell proliferation (<strong role="strong">Figure</strong>  1.25 <a href="#Bio08-04-04-fig" id="Bio08-04-04-text">below</a>).</p><h4>Oncogenes</h4><p>The products of proto-oncogenes are required for normal growth, repair and homeostasis. However, when these genes are mutated, they turn into oncogenes and play a role in the development of cancer. Proto-oncogenes may be growth factors, transcription factors, or other proteins involved in regulation. A very common oncogene, ras, is normally a regulatory GTPase that switches a signal transduction chain on and off. Ras and Ras-related proteins are products of oncogenes found in 20% to 30% of human tumors.</p><p>Ras is a <strong role="strong">G protein</strong>, a regulatory GTP hydrolase that cycles between an activated and inactivated form. When a growth factor binds to its receptor on the outside of the cell, a signal is relayed to RAS. As a G protein, Ras is activated when GTP is bound to it. The active Ras then passes the signal to a series of protein kinases, regulatory proteins that eventually activate transcription factors to alter gene expression and produce proteins that stimulate the cell cycle (<strong role="strong">Figure</strong>  1.25 <a href="#Bio08-04-04-fig" id="Bio08-04-04-text">below</a>). Many of the genes and proteins involved in signal transduction pathways are interconnected to ras. Any mutation that makes ras more active or otherwise interrupts the normal signal transduction pathways (<strong role="strong">Figure</strong>  1.25 <a href="#Bio08-04-04-fig" id="Bio08-04-04-text">below</a>) may result in excessive cell division and cancer.</p><h4>Tumor Suppressor Genes</h4><p>An example of a tumor suppressor gene is p53, which encodes a 53,000 dalton protein, The p53 gene is activated by DNA damage. DNA may be damaged by ultraviolet light, and any damaged DNA may be harmful to the cell. Mutations causing problems with any of the components of <strong role="strong">Figure</strong>  1.25 <a href="#Bio08-04-04-fig" id="Bio08-04-04-text">below</a> may lead to the development of cancer. So that damaged DNA is not replicated, the cell cycle must be temporarily stopped so that the DNA can be repaired. The p53 tumor suppressor gene encodes a transcription factor that regulates the synthesis of cell cycle inhibiting proteins (<strong role="strong">Figure</strong>  1.25 <a href="#Bio08-04-04-fig" id="Bio08-04-04-text">below</a>). p53 often activates a gene named p21, whose protein product temporarily stops the cell cycle. If the DNA can not be repaired, p53 activates other genes that lead to cell death, or apoptosis. This prevents the cell from passing on damaged DNA. If the p53 tumor suppressor gene is defective, as by mutation, DNA damage in the cell may accumulate and the cell may survive to replicate the damaged DNA. The damaged DNA would then be passed to other cells through many cell divisions, and cancer could develop.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2665" width="256px" /></div> <p><a href="#Bio08-04-04-text" id="Bio08-04-04-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.25 <caption>Signal transduction pathways. Ras (upper middle section) activates a number of pathways but an especially important one seems to be the mitogen-activated protein kinases (MAPK). MAPK transmit signals downstream to other protein kinases and gene regulatory proteins. Note that many of these pathways are initiated when a signal binds to its receptor outside the cell. Most pathways end with altered gene regulation and cell proliferation. The p53 tumor suppressor protein is shown at the lower section of the figure stimulating p21. The complexity of the pathways demonstrate the significant role these play in the cell.</caption></p></div><p> </p><h3>Lesson Summary</h3><ul><li>A cis-regulatory element is a region of DNA which regulates the expression of a gene or multiple genes located on that same strand of DNA. </li><li>The cis-regulatory elements are often binding sites for one or more trans-acting factors, usually a regulatory protein which interacts with RNA polymerase. </li><li>Repressor proteins bind to non-coding sequences on the DNA that are close to or overlap the promoter region, impeding RNA polymerase's progress along the strand. </li><li>Enhancers are sites on the DNA strand that are bound by activators. </li><li>Prokaryotic genes are arranged in operons, regions of DNA with a promoter, an operator, and one or more genes that encode proteins needed to perform a certain task. </li><li>The regulation of the lac operon is a key example of prokaryotic gene regulation. When lactose is present, RNA polymerase binds to the promoter and the operon is turned on; when lactose is unavailable, the lac repressor binds to the operator and the operon is turned off. </li><li>Homeobox genes are involved in the regulation of genes important to development. They encode transcription factors. </li><li>Gap genes control the shape of a developing zygote early in its development. </li><li>At least two separate mutations are necessary to develop cancer. These mutations may occur in proto-oncogenes and/or tumor suppressor genes. </li></ul><h3>Review Questions</h3><ol start><li>What is meant by gene expression, and why is this an important cellular mechanism?</li><li>How do cis-regulatory elements and a trans-acting factors work together?</li><li>Define a transcription factor.</li><li>Define an operon. Give an example.</li><li>Describe how the lac operon regulates transcription.</li><li>Describe the role of the TATA box.</li><li>Why is gene regulation an important aspect of development?</li><li>What is a homeobox gene? A gap gene? Why are these genes important?</li><li>Why does altered gene regulation have a potential role in the development of cancer?</li></ol><h3>Further Reading / Supplemental Links</h3><ul><li>Campbell, N.A. and Reece, J.B. <em>Biology</em>, Seventh Edition, Benjamin Cummings, San Francisco, CA, 2005. </li><li>Biggs, A., Hagins, W.C., Kapicka, C., Lundgren, L., Rillero, P., Tallman, K.G., and Zike, D., <em>Biology: The Dynamics of Life</em>, California Edition, Glencoe Science, Columbus, Oh, 2005. </li><li>Nowicki S., <em>Biology</em>, McDougal Littell, Evanston, IL, 2008.</li><li><a href="http://en.wikipedia.org"> </a><a href="http://en.wikipedia.org">http://en.wikipedia.org</a></li></ul><h3>Vocabulary</h3><dl><dt>activator </dt><dd>Protein that enhances the interaction between RNA polymerase and a particular promoter. </dd></dl><dl><dt>basal factor </dt><dd>Transcription factor that helps position RNA polymerase at the start of a gene. </dd></dl><dl><dt>cis-regulatory element </dt><dd>A region of DNA which regulates the expression of a gene or multiple genes located on that same strand of DNA. </dd></dl><dl><dt>enhancer </dt><dd>Site on the DNA strand that can be bound by activator(s) in order to loop the DNA, bringing a specific promoter to the initiation complex. </dd></dl><dl><dt>G protein </dt><dd>A regulatory GTP hydrolase that cycles between an activated and inactivated form; activated when GTP is bound to it. </dd></dl><dl><dt>gap gene </dt><dd>Gene that controls the shape of a developing zygote early in its development; encodes transcription factors. </dd></dl><dl><dt>homeobox </dt><dd>A 180 base pair long highly conserved segment of DNA; encodes a 60 amino acid domain within the protein (known as the homeodomain), which can bind DNA. </dd></dl><dl><dt>homeobox genes </dt><dd>Genes that contain a highly conserved DNA sequence known as a homeobox and are involved in the regulation of genes important to development; encodes transcription factors. </dd></dl><dl><dt>hox genes </dt><dd>Genes that function in patterning the body, providing the placement of certain body parts during development. </dd></dl><dl><dt>initiation complex </dt><dd>Complex needed to start transcription in eukaryotes; composed of RNA polymerase and trans-acting factors. </dd></dl><dl><dt>lac operon </dt><dd>An operon required for the transport and metabolism of lactose in <em>E. coli</em>.</dd></dl><dl><dt>operator </dt><dd>A region of prokaryotic DNA where a repressors binds. </dd></dl><dl><dt>operon </dt><dd>A region of prokaryotic DNA with a promoter, an operator, and one or more genes that encode proteins needed to perform a certain task. </dd></dl><dl><dt>promoter </dt><dd>A segment of DNA that allows a gene to be transcribed; helps RNA polymerase find the start of a gene. </dd></dl><dl><dt>repressor </dt><dd>Protein that binds to non-coding sequences on the DNA that are close to or overlap the promoter region, impeding RNA polymerase's progress along the strand. </dd></dl><dl><dt>RNA polymerase </dt><dd>The enzyme that transcribes the DNA to make RNA. </dd></dl><dl><dt>specificity factor </dt><dd>Protein that alters the specificity of RNA polymerase for a promoter or set of promoters, making it more or less likely to bind to the promoter and begin transcription. </dd></dl><dl><dt>TATA box </dt><dd>A <em>cis</em>-regulatory element found in the promoter of most eukaryotic genes; when the appropriate cellular signals are present, RNA polymerase binds to the TATA box, completing the initiation complex.</dd></dl><dl><dt>transcription factor </dt><dd>A protein involved in regulating gene expression; usually bound to a cis-regulatory element; also known as a regulatory protein. </dd></dl><dl><dt>zinc finger </dt><dd>A DNA binding region within certain proteins encoded by a gap gene; consists of two antiparallel &beta; sheets and an &alpha; helix with a zinc ion, which is important for the stability of this region. </dd></dl><h3>Points to Consider</h3><ul><li>The next chapter is <em>Human Genetics</em>. Discuss why an understanding of human genetics is an important medical issue for our society. </li><li>We have extensively discussed mutations and cancer. There are many other phenotypes due to mutations in the human genome. Why is understanding mutations in humans important? </li><li>What do you think the Human Genome Project is? What could some implications of The Human Genome Project be? </li></ul>


