<h2>Lesson 10.1: DNA Technology</h2><h3>Lesson Objectives</h3><ul><li>What is meant by DNA technology? </li><li>What is the Human Genome Project? </li><li>Describe the goals of the Human Genome Project. </li><li>Describe gene cloning and the processes involved. </li><li>What is PCR? </li><li>Describe the processes involved in PCR. </li></ul><h3>Introduction</h3><p>Is it really possible to clone people? Another question is, should we clone people? Are scientific fantasies, such as depicted on TV shows such as <em>Star Trek</em> or in the movie <em>GATTACA</em>, actually a possibility? Who can really say? How, really, will science affect our future? The answers partially lie in the field of biotechnology.</p><p><strong role="strong">Biotechnology</strong> is technology based on biological applications. These applications are increasingly used in medicine, agriculture and food science. Biotechnology combines many features of biology, including genetics, molecular biology, biochemistry, embryology, and cell biology. Many aspects of biotechnology center around DNA and its applications, otherwise known as DNA technology. We could devote a whole textbook to current applications of biotechnology; in this chapter, however, we will focus on the applications towards medicine and the extension into the forensic sciences. First, though, we need to understand DNA technology.</p><h3>DNA Technology</h3><p>What is DNA technology? Is it using and manipulating DNA to help people? Is it using DNA to make better medicines and individualized treatments? Is it analyzing DNA to determine predispositions to genetic diseases? The answers to these questions, and many more, is yes. And the answers to many of these issues begin with the Human Genome Project.</p><h3>The Human Genome Project</h3><p>If we are all 99.9% genetically identical, what makes us different? How does that 0.1% make us tall or short, light or dark, develop cancer or not? To understand that 0.1%, we also need to understand the other 99.9%. Understanding the human genome is the goal of <strong role="strong">The Human Genome Project (HGP)</strong>. This project, publicly funded by the United States Department of Energy (DOE) (<strong role="strong">Figure</strong>  1.1 <a href="#Bio101-01-fig" id="Bio101-01-text">below</a>); and the National Human Genome Research Institute (NHGRI), part of the National Institutes of Health (NIH), may be one of the landmark scientific events of our lifetime.  <em>Our Molecular Selves</em> video discusses the human genome, and is available at <a href="http://www.genome.gov/25520211.">http://www.genome.gov/25520211.</a></p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2712" width="256px" /></div> <p><a href="#Bio101-01-text" id="Bio101-01-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.1 <caption>The Human Genome Project logo of the DOE.</caption></p></div><p> </p><p>The goal of the HGP is to understand the genetic make-up of the human species by determining the DNA sequence of the human genome (<strong role="strong">Figure</strong>  1.2 <a href="#Bio10-1-2-fig" id="Bio10-1-2-text">below</a>);and the genome of a few model organisms. However, it is not just determining the 3 billion bases; it is understanding what they mean. Today, all 3 billion base pairs have been sequenced, and the genes in that sequence are in the process of being identified and characterized. A preliminary estimate of the number of genes in the human genome is around 22,000 to 23,000.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2710" width="256px" /></div> <p><a href="#Bio10-1-2-text" id="Bio10-1-2-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.2 <caption>A depiction of DNA sequence analysis. Note the 4 colors utilized, each representing a separate base.</caption></p></div><p> </p><p>The sequence of the human DNA is stored in databases available to anyone on the Internet. The U.S. National Center for Biotechnology Information (NCBI), part of the NIH, as well as comparable organizations in Europe and Japan, maintain the genomic sequences in a database known as <strong role="strong">Genbank</strong>. Protein sequences are also maintained in this database. The sequences in these databases are the combined sequences of anonymous donors, and as such do not yet address the individual differences that make us unique. However, the known sequence does lay the foundation to identify the unique differences among all of us. Most of the currently identified variations among individuals will be <strong role="strong">single nucleotide polymorphisms</strong>, or SNPs. A <strong role="strong">SNP</strong> (pronounced "snip") is a DNA sequence variation occurring at a single nucleotide in the genome. For example, two sequenced DNA fragments from different individuals, GGATCTA to GGATTTA, contain a difference in a single nucleotide. If this base change occurs in a gene, the base change then results in two alleles: the C allele and the T allele. Remember an allele is an alternative form of a gene. Almost all common SNPs have only two alleles. The effect of these SNPs on protein structure and function, and any effect on the resulting phenotype, is an extensive field of study.</p><h3>Gene Cloning</h3><p>You probably have heard of cloning. Whereas cloning of humans has many ethical issues associated with it, the cloning of genes has been ongoing for well over 30 years, with cloning of animals occurring more recently. <strong role="strong">Gene cloning</strong>, also known as molecular cloning, refers to the process of isolating a DNA sequence of interest for the purpose of making multiple copies of it. The identical copies are clones. In 1973, Stanley Cohen and Herbert Boyer developed techniques to make recombinant DNA, a form of artificial DNA.</p><p><strong role="strong">Recombinant DNA</strong> is engineered through the combination of two or more DNA strands, combining DNA sequences which would not normally occur together. In other words, selected DNA (or the DNA of "interest") is inserted into an existing organismal genome, such as a bacterial plasmid DNA or some other sort of vector. The recombinant DNA can then be inserted into another cell, such as a bacterial cell, for amplification and possibly production of the resulting protein. This process is called <strong role="strong">transformation</strong>, the genetic alteration of a cell resulting from the uptake, incorporation, and expression of foreign genetic material. Recombinant DNA technology was made possible by the discovery of restriction endonucleases.</p><h4>Restriction Enzyme Digestion and Ligation</h4><p>In the classical <strong role="strong">restriction enzyme</strong> digestion and ligation cloning protocols, cloning of any DNA fragment essentially involves four steps:</p><ol start><li>isolation of the DNA of interest (or target DNA) </li><li>ligation </li><li>transfection (or transformation) </li><li>a screening/selection procedure. </li></ol><p>For an overview of cloning, see <a href="http://www.hhmi.org/biointeractive/media/DNAi_genetic_eng-sm.mov.">http://www.hhmi.org/biointeractive/media/DNAi_genetic_eng-sm.mov.</a></p><h5>Isolation of DNA</h5><p>Initially, the DNA fragment to be cloned needs to be isolated. This DNA of interest may be a gene, part of a gene, a promoter, or another segment of DNA, and is frequently isolated by the Polymerase Chain Reaction (PCR) or restriction enzyme digestion. A <strong role="strong">restriction enzyme</strong> (or <strong role="strong">restriction endonuclease</strong>) is an enzyme that cuts double-stranded DNA at a specific sequence (<strong role="strong">Table</strong>  1.1 <a href="#table:specific sequence enzyme-fig" id="table:specific sequence enzyme-text">below</a>). The enzyme makes two incisions, one through each strand of the double helix, without damaging the nitrogenous bases. This produces either overlapping ends (also known as sticky ends) or blunt ends.</p><div><p><strong>Table </strong>1.1 </p><table><p></p><thead><tr><td>A.</td><td>G<sup>&darr;</sup>AATTC</td><td>B.</td><td>CCC<sup>&darr;</sup>GGG</td></tr></thead><tbody><tr><td> </td><td>CTTAA<sub>&uarr;</sub>G</td><td> </td><td>GGG<sub>&uarr;</sub>CCC</td></tr></tbody></table></div><p>A. <em>Eco</em>RI digestion produces overlapping "sticky" ends: The enzyme cleaves between the G and A on both strands. B. <em>Sma</em>II restriction enzyme cleavage produces "blunt" ends. The enzyme cleaves between the G and C on both strands.</p><p>(<em>Source:</em> <em>Created by:</em> Doug Wilkin, <em>License:</em> CC-BY-SA)</p><p>The 1978 Nobel Prize in Medicine was awarded to Daniel Nathans and Hamilton Smith for the discovery of restriction endonucleases. The first practical use of their work was the manipulation of <em>E. coli</em> bacteria to produce human insulin for diabetics.</p><h5>Ligation</h5><p>Once the DNA of interest is isolated, a ligation procedure is necessary to insert the amplified fragment into a vector to produce the recombinant DNA molecule. Restriction fragments (or a fragment and a plasmid/vector) can be spliced together, provided their ends are complementary. Blunt end ligation is also possible.</p><p>The <strong role="strong">plasmid</strong> or vector (which is usually circular) is digested with restriction enzymes, opening up the vector to allow insertion of the target DNA. The two DNAs are then incubated with <strong role="strong">DNA ligase</strong>, an enzyme that can attach together strands of DNA with double strand breaks. This produces the recombinant DNA molecule. <strong role="strong">Figure</strong>  1.3 <a href="#Bio101-04-fig" id="Bio101-04-text">below</a> depicts a plasmid with two additional segments of DNA ligated into the plasmid, producing the recombinant DNA molecule. <strong role="strong">Figure</strong>  1.4 <a href="#Bio101-05b-fig" id="Bio101-05b-text">below</a> depicts DNA before and after ligation.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2702" width="256px" /></div> <p><a href="#Bio101-04-text" id="Bio101-04-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.3 <caption>This image shows a line drawing of a plasmid. The plasmid is drawn as two concentric circles that are very close together, with two large segments and one small segment depicted. The two large segments (1 and 2) indicate antibiotic resistances usually used in a screening procedure, and the small segment (3) indicates an origin of replication. The resulting DNA is a recombinant DNA molecule.</caption></p></div><p> </p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2701" width="256px" /></div> <p><a href="#Bio101-05b-text" id="Bio101-05b-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.4 <caption>Sticky ends produced by restriction enzyme digestion can be joined with the enzyme DNA ligase.</caption></p></div><p> </p><h5>Transfection and Selection</h5><p>Following ligation, the recombinant DNA is placed into a host cell, usually bacterial, in a process called <strong role="strong">transfection</strong> or <strong role="strong">transformation</strong>. Finally, the transfected cells are cultured. Many of these cultures may not contain a plasmid with the target DNA as the transfection process is not usually 100% successful, so the appropriate cultures with the DNA of interest must be selected. Many plasmids/vectors include selectable markers - usually some sort of antibiotic resistance (<strong role="strong">Figure</strong>  1.3 <a href="#Bio101-04-fig" id="Bio101-04-text">above</a>). When cultures are grown in the presence of an antibiotic, only bacteria transfected with the vector containing resistance to that antibiotic should grow. However, these selection procedures do not guarantee that the DNA of insert is present in the cells. Further analysis of the resulting colonies is required to confirm that cloning was successful. This may be accomplished by means of a process known as PCR (see below) or restriction fragment analysis, both of which need to be followed by gel electrophoresis and/or DNA sequencing (DNA sequence analysis).</p><p>DNA sequence analysis (the analysis of the order of the nitrogenous bases that make up the DNA), PCR, or restriction fragment analysis will all determine if the plasmid/vector contains the insert. Restriction fragment analysis is digestion of isolated plasmid/vector DNA with restriction enzymes. If the isolated DNA contains the target DNA, that fragment will be excised by the restriction enzyme digestion. Gel electrophoresis will separate DNA molecules based on size and charge. Examples are shown in <strong role="strong">Figure</strong>  1.5 <a href="#Bio10-01-06-fig" id="Bio10-01-06-text">below</a>.</p><h5>Gel Electrophoresis</h5><p><strong role="strong">Gel electrophoresis</strong> is an analytical technique used to separate DNA fragments by size and charge. Notice in <strong role="strong">Figure</strong>  1.5 <a href="#Bio10-01-06-fig" id="Bio10-01-06-text">below</a> that the "gels" are rectangular in shape. The gels are made of a gelatin-like material of either agarose or polyacrylamide. An electric field, with a positive charge applied at one end of the gel, and a negative charge at the other end, forces the fragments to migrate through the gel. DNA molecules migrate from negative to positive charges due to the net negative charge of the phosphate groups in the DNA backbone. Longer molecules migrate more slowly through the gel matrix. After the separation is completed, DNA fragments of different lengths can be visualized using a fluorescent dye specific for DNA, such as ethidium bromide. The resulting stained gel shows bands correspond to DNA molecules of different lengths, which also correspond to different molecular weights. Band size is usually determined by comparison to DNA ladders containing DNA fragments of known length. Gel electrophoresis can also be used to separate RNA molecules and proteins.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2711" width="256px" /></div> <p><a href="#Bio10-01-06-text" id="Bio10-01-06-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.5 <caption>Agarose gel following agarose gel electrophoresis on UV light box: In the gel with UV illumination (left), the ethidium bromide stained DNA glows pink; Right, photo of a gel. Far left: DNA ladder of fragments of known length. Lane 1: A PCR product of just over 500 bases. Lane 2: Restriction digest showing the 500 base fragment cut from a 4.5 kb plasmid vector.</caption></p></div><p> </p><h3>The Polymerase Chain Reaction</h3><p>The <strong role="strong">Polymerase Chain Reaction</strong> (PCR) is used to amplify specific regions of a DNA strand millions of times. A region may be a number of loci, a single gene, a part of a gene, or a non-coding sequence. This technique produces a useful quantity of DNA for analysis, be it medical, forensic or some other form of analysis. Amplification of DNA from as little as a single cell is possible. Whole genome amplification is also possible.</p><p>PCR utilizes a heat stable DNA polymerase, <strong role="strong">Taq polymerase</strong>, named after the thermophilic bacterium <em>Thermus aquaticus,</em> from which it was originally isolated. <em>T. aquaticus</em> is a bacterium that lives in hot springs and hydrothermal vents, and Taq polymerase is able to withstand the high temperatures required to denature DNA during PCR (discussed below). Taq polymerase&rsquo;s optimum temperature for activity is between 75&deg;C and 80&deg;C. Recently other DNA polymerases have also been used for PCR.</p><p>A basic PCR involves a series of repeating cycles involving three main steps (see <strong role="strong">Figure</strong>  1.6 <a href="#Bio_PCR-fig" id="Bio_PCR-text">below</a>):</p><ol start><li>denaturation of the double stranded DNA </li><li>annealing of specific oligonucleotide primers </li><li>extension of the primers to amplify the region of DNA of interest </li></ol><p>These steps will be discussed in additional detail below.</p><p>The oligonucleotide primers are single stranded pieces of DNA that correspond to the 5&rsquo; and 3&rsquo; ends of the DNA region to be amplified. These primers will anneal to the corresponding segment of denatured DNA. Taq Polymerase, in the presence of free deoxynucleotide triphosphates (dNTPs), will extend the primers to create double stranded DNA. After many cycles of denaturation, annealing and extension, the region between the two primers will be amplified.</p><p>The PCR is commonly carried out in a thermal cycler, a machine that automatically allows heating and cooling of the reactions to control the temperature required at each reaction step (see below). The PCR usually consists of a series of about 30 to 35 cycles. Most commonly, PCR is carried out in three repeating steps, with some modifications for the first and last step.</p><p>PCR is usually performed in small tubes or wells in a tray, each often beginning with the complete genome of the species being studied.  As only a specific sequence from that genome is of interest, the sequence specific primers are targeted to that sequence. PCR is done with all the building blocks necessary to create DNA: template DNA, primers, dNTPs, and a polymerase.</p><p>The three basic steps of PCR (<strong role="strong">Figure</strong>  1.6 <a href="#Bio_PCR-fig" id="Bio_PCR-text">below</a>) are:</p><ul><li>Denaturation step: This step is the first regular cycling event and consists of heating the reaction to 94 - 98&deg;C for 30 to 60 seconds. It disrupts the hydrogen bonds between complementary bases of the DNA strands, yielding single strands of DNA. </li><li>Annealing step: The reaction temperature is lowered to 50-65&deg;C for 30 to 60 seconds, allowing annealing of the primers to the single-stranded DNA template. Stable hydrogen bonds form between the DNA strand (the template) and the primers when the primer sequence very closely matches the complementary template sequence. Primers are usually 17 - 22 nucleotides long and are carefully designed to bind to only one site in the genome. The polymerase binds to the primer-template hybrid and begins DNA synthesis. </li><li>Extension step: A temperature of around 72&deg;C is used for this step, which is close to the optimum temperature of Taq polymerase. At this step the Taq polymerase extends the primer by adding dNTPs, using one DNA strand as a template to create a the other (new) DNA strand. The extension time depends on the length of the DNA fragment to be amplified. As a standard, at its optimum temperature, the DNA polymerase will polymerize a thousand bases in one minute. </li></ul><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2708" width="256px" /></div> <p><a href="#Bio_PCR-text" id="Bio_PCR-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.6 <caption>PCR: A repeating cycle of denaturation (1), annealing (2), and extension (3). Notice that initially there is a double strand of DNA, and after denaturation, the DNA is single stranded.  In the annealing step (2), single stranded primers bind.  These primers are extended by Taq Polymerase, represented by the green ball (3).</caption></p></div><p> </p><p>Utilizing the PCR, DNA can be amplified millions of times to generate quantities of DNA that can be used for a number of purposes. These include the use of DNA for prenatal or genetic testing, such as testing for a specific mutation. PCR has revolutionized the fields of biotechnology, human genetics, and a number of other sciences. Many of the applications will be discussed in the following lesson. PCR was developed in 1983 by Kary Mullis. Due to the importance of this process and the significance it has had on scientific research, Dr. Mullis was awarded the Nobel Prize in Chemistry in 1993, just 10 years after his discovery.</p><p>To say that PCR, molecular cloning and the Human Genome Project has revolutionized biology and medicine would be an understatement. These efforts have led to numerous accolades, including Nobel prizes, and more may follow. Some of the ways that these discoveries have shaped our lives are the focus of the next lesson.</p><h3>Lesson Summary</h3><ul><li>Biotechnology is technology based on biological applications, combining many features of Biology including genetics, molecular biology, biochemistry, embryology, and cell biology. </li><li>The goal of the Human Genome Project is to understand the genetic make-up of the human species by determining the DNA sequence of the human genome and the genome of a few model organisms. </li><li>Gene cloning, also known as molecular cloning, refers to the process of isolating a DNA sequence of interest for the purpose of making multiple copies of it (cloning). </li><li>Classic gene cloning involves the following steps: </li></ul><ol start><li>Restriction enzyme digestion and ligation </li><li>Isolation of DNA </li><li>Ligation </li><li>Transfection and Selection </li><li>Gel electrophoresis </li></ol><ul><li>The Polymerase Chain Reaction (PCR) is used to amplify millions of times specific regions of a DNA strand. This can be a number of loci, a single gene, a part of a gene, or a non-coding sequence. </li><li>PCR usually involves the following steps: </li></ul><ol start><li>Denaturation step </li><li>Annealing step </li><li>Extension step </li></ol><h3>Review Questions</h3><ol start><li>Why are biotechnology and DNA technology considered the same?</li><li>What are the goals of the Human Genome Project?</li><li>Is the DNA sequence information generated by the HGP available to anyone, and if so, how?</li><li>How are gene cloning and recombinant DNA related?</li><li>Describe the process of gene cloning.</li><li>How does gel electrophoresis analyze DNA?</li><li>What is PCR?</li><li>What allows PCR to be done at high temperatures?</li><li>Describe the PCR process.</li></ol><h3>Further Reading / Supplemental Links</h3><ul><li><a href="http://www.genome.gov/HGP"> </a><a href="http://www.genome.gov/HGP">http://www.genome.gov/HGP</a></li><li><a href="http://www.ornl.gov/sci/techresources/Human_Genome/home.shtml"> </a><a href="http://www.ornl.gov/sci/techresources/Human_Genome/home.shtml">http://www.ornl.gov/sci/techresources/Human_Genome/home.shtml</a></li><li><a href="http://biotech.about.com/od/cloning/tp/DNAcloning.htm"> </a><a href="http://biotech.about.com/od/cloning/tp/DNAcloning.htm">http://biotech.about.com/od/cloning/tp/DNAcloning.htm</a></li><li><a href="http://croptechnology.unl.edu/viewLesson.cgi?LessonID=957884601"> </a><a href="http://croptechnology.unl.edu/viewLesson.cgi?LessonID=957884601">http://croptechnology.unl.edu/viewLesson.cgi?LessonID=957884601</a></li><li><a href="http://www.pcrstation.com/"> </a><a href="http://www.pcrstation.com/">http://www.pcrstation.com/</a></li><li><a href="http://nobelprize.org/educational_games/chemistry/pcr/index.html"> </a><a href="http://nobelprize.org/educational_games/chemistry/pcr/index.html">http://nobelprize.org/educational_games/chemistry/pcr/index.html</a></li><li><a href="http://en.wikipedia.org"> </a><a href="http://en.wikipedia.org">http://en.wikipedia.org</a></li><li><a href="http://www.vtaide.com/png/cloning.htm">http://www.vtaide.com/png/cloning.htm</a></li></ul><h3>Vocabulary</h3><dl><dt>biotechnology </dt><dd>Technology based on biological applications. </dd></dl><dl><dt>gel electrophoresis </dt><dd>An analytical technique used to separate DNA fragments by size and charge. </dd></dl><dl><dt>Genbank </dt><dd>The U.S. National Center for Biotechnology Information, part of the National Institutes of Health, which maintains the genomic sequences in a database. </dd></dl><dl><dt>gene cloning </dt><dd>The process of isolating a DNA sequence of interest for the purpose of making multiple copies of it. </dd></dl><dl><dt>The Human Genome Project </dt><dd>A project to understand the genetic make-up of the human species by determining the DNA sequence of the human genome and the genome of a few model organisms. </dd></dl><dl><dt>plasmid (or vector) </dt><dd>A small circular piece of DNA that carries the recombinant DNA into a host organism for cloning. </dd></dl><dl><dt>polymerase chain reaction (PCR) </dt><dd>A repeating series of cycles used to amplify millions of times specific regions of a DNA strand. </dd></dl><dl><dt>recombinant DNA </dt><dd>Engineered through the combination of two or more DNA strands that combine DNA sequences which would not normally occur together. </dd></dl><dl><dt>restriction enzyme (or restriction endonuclease) </dt><dd>An enzyme that cuts double-stranded DNA. </dd></dl><dl><dt>Taq polymerase </dt><dd>Named after the thermophilic bacterium <em>Thermus aquaticus</em> from which it was originally isolated, is the heat-stable polymerase used in the PCR reaction.</dd></dl><h3>Points to Consider</h3><ul><li>The Human Genome Project, gene cloning, and PCR are some of the most remarkable scientific achievements of the recent past. But how can these milestones make our lives better? </li><li>Medicine and food science are just two of the categories that benefit from biotechnology. Speculate on how our lives are made better by these achievements. </li></ul>