<h2>Lesson 13.2: Genetic Change in Populations</h2><h3>Lesson Objectives</h3><ul><li>Compare and relate macroevolution to microevolution. </li><li>Define microevolution in terms of allele frequencies. </li><li>Define genetic equilibrium for a population. </li><li>List the five conditions for genetic equilibrium according to the Hardy-Weinberg model. </li><li>State and explain the generalized equation for Hardy-Weinberg equilibrium. </li><li>Explain how to use the Hardy-Weinberg equation to solve for allele or genotype frequencies. </li><li>Discuss the reasons why the 5 conditions for Hardy-Weinberg equilibrium are rarely met. </li><li>Explain how mutation disrupts genetic equilibrium. </li><li>Predict the possible effects of mutation, and analyze the probability of each type of effect. </li><li>Contrast mutation in microorganisms to mutation in multicellular organisms. </li><li>Define gene flow. </li><li>Describe two possible effects of gene flow on the genetics of a population. </li><li>Define genetic drift. </li><li>Describe three possible effects of genetic drift on populations and/or specific alleles. </li><li>Explain and give an example of the bottleneck effect. </li><li>Clarify and give an example of the concept of &ldquo;founder effect.&rdquo; </li><li>Compare and contrast genetic drift and natural selection as causes of evolution. </li><li>Discuss natural selection and evolution in terms of phenotypes and allele frequencies. </li><li>Explain the distribution of phenotypes for a trait whose genetic basis is polygenic. </li><li>Using a normal distribution of phenotypic variation, interpret directional, disruptive, and stabilizing patterns of selection. </li><li>Define fitness as it relates to natural selection. </li><li>Describe how natural selection can sometimes lead to the persistence of harmful or even lethal allele. </li><li>Analyze the logic of kin selection. </li></ul><h3>Introduction</h3><ul><li>How many people in the United States carry the recessive allele for the lethal genetic disease, cystic fibrosis? </li><li>Why must we treat AIDS patients with multi-drug &ldquo;cocktails?&rdquo; </li><li>Why do we consider Northern Elephant seals endangered, even though their population has risen to 100,000 individuals? </li><li>Why don&rsquo;t South African Cheetahs reject skin grafts from unrelated individuals? </li><li>What were the probable effects on human evolution of the six years of &ldquo;volcanic winter&rdquo; caused by the &ldquo;megacollosal&rdquo; eruption of Mt. Toba 70,000 years ago? </li><li>Why do humans vary in skin color? </li><li>Why doesn&rsquo;t natural selection eliminate certain lethal genes? </li></ul><p>This lesson will introduce you to our current understanding of the causes of evolution and show you how scientists use this understanding to solve problems and answer puzzling questions like those above.</p><h3>Microevolution</h3><p>In an earlier chapter, we defined <strong role="strong">microevolution</strong> as evolution within a species or population, and <strong role="strong">macroevolution</strong> as evolution at or above the level of species. The differences between the two are time and scale. Microevolution can refer to changes as small as a shift in allele frequencies for a single gene from one generation to the next. In contrast, macroevolution describes changes in species over geologic time. Many biologists consider evolution to be a single process; they regard macroevolution as the cumulative effects of microevolution. The <em>History of Life</em> chapter discussed patterns and processes of macroevolution; this lesson will focus on genetic changes within populations and species. At this level, we can see how evolution really works &ndash; a view that Darwin never had.</p><p>A change in allele frequencies within a population from one generation to the next &ndash; even if it involves only a single gene - is evolution. Biologists refer to this change in the gene pool as microevolution because it is evolution on the smallest scale. For our rabbit population from the previous lesson, a generation-to-generation increase in the frequency of the albino allele, <em>b,</em> from 0.3 to 0.4 (and the corresponding decrease in the frequency of allele <em>B</em> from 0.7 to 0.6) would be microevolution. The changes in Galapagos finch beak size, documented by Peter and Rosemary Grant, and the color changes of peppered moths will undoubtedly show corresponding changes in allele frequencies once we identify the responsible genes and alleles. Resistance to antibiotics in bacteria and the appearance of new strains of influenza viruses and HIV <strong role="strong">Figure</strong>  1.9 <a href="#Bio132-01-fig" id="Bio132-01-text">below</a> also involve changes in allele frequencies. Each of these is an example of microevolution.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2806" width="256px" /></div> <p><a href="#Bio132-01-text" id="Bio132-01-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.9 <caption>This portion of the evolutionary tree for Human Immunodeficiency Virus (HIV) shows 8 or more strains of the HIV-1 M (Main) group, and a single strain of the HIV-1 N (Not Main/Not Outlier) group. The complete tree includes strains of SIV (Simian Immunodeficiency Virus). Each strain represents a change in allele frequencies.</caption></p></div><p> </p><p>What forces cause changes in allele frequencies? What mechanisms determine how microevolution happens? Before we tackle these questions, we will examine a population at equilibrium &ndash; a population which is <em>not</em> evolving.</p><h3>Populations at Equilibrium: The Hardy-Weinberg Model</h3><p>Like Darwin and Wallace, who independently developed similar ideas about evolution and natural selection, mathematician Godfrey Hardy and physician Wilhelm Weinberg independently developed a model of populations at equilibrium. The <strong role="strong">Hardy-Weinberg model</strong> (sometimes called a law) states that a population will remain at genetic equilibrium - with constant allele and genotype frequencies and no evolution - as long as five conditions are met:</p><ol start><li>No mutation </li><li>No migration </li><li>Very large population size </li><li>Random mating </li><li>No natural selection </li></ol><p>We will consider the above in more detail in later sections of the lesson, because deviations from these conditions are the causes of evolutionary change. For now, let&rsquo;s look more closely at the Hardy-Weinberg&rsquo;s equilibrium model. We&rsquo;ll use another hypothetical rabbit population to make the model concrete: 9 albino rabbits and 91 brown rabbits (49 homozygous and 42 heterozygous). The gene pool contains 140 <em>B</em> alleles and 60 <em>b</em> alleles &ndash; gene frequencies of 0.7 and 0.3, respectively. <strong role="strong">Figure</strong>  1.10 <a href="#Bio13-2-t1-fig" id="Bio13-2-t1-text">below</a> summarizes the data for this parent population.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2818" width="256px" /></div> <p><a href="#Bio13-2-t1-text" id="Bio13-2-t1-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.10 <caption>The coat color gene pool for a hypothetical rabbit population includes two alleles. Genotype and allele frequencies are calculated, given the number of individuals having each of the three possible genotypes.</caption></p></div><p> </p><p>If we assume that alleles sort independently and segregate randomly as sperm and eggs form, and that mating and fertilization are also random, the probability that an offspring will receive a particular allele from the gene pool is identical to the frequency of that allele in the population. A Punnett Square (<strong role="strong">Figure</strong>  1.11 <a href="#Bio13-2-t2-fig" id="Bio13-2-t2-text">below</a>) using these frequencies predicts the probability of each genotype (and phenotype) in the next generation:</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2809" width="256px" /></div> <p><a href="#Bio13-2-t2-text" id="Bio13-2-t2-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.11 <caption>A Punnett Square predicts the probability of each genotype and phenotype for the offspring of the population described in  . A summarizes the frequencies of each genotype among offspring, and B calculates allele frequencies for the next generation. Comparing these to the parent generation shows that the gene pool remains constant. The population is stable  at a Hardy-Weinberg equilibrium.</caption></p></div><p> </p><p>If we calculate the frequency of genotypes among the offspring predicted by the Punnett square, they are identical to the genotype frequencies of the parents. Allele frequency remains constant as well. The population is stable &ndash; at a Hardy-Weinberg <strong role="strong">genetic equilibrium.</strong></p><p>A useful equation generalizes the calculations we&rsquo;ve just completed. Variables include <strong role="strong"><em>p</em></strong> = the frequency of one allele (we&rsquo;ll use allele <strong role="strong"><em>B</em></strong> here) and <strong role="strong"><em>q</em></strong> = the frequency of the second allele (<strong role="strong"><em>b</em></strong>, in this example). We will use only two alleles, but similar, valid equations can be written for more alleles.</p><p>Recall that the allele frequency equals the probability of any particular gamete receiving that allele. Therefore, when egg and sperm combine, the probability of any genotype (as in the Punnett square above) is the product of the probabilities of the alleles in that genotype. So:</p><p>Probability of genotype <strong role="strong"><em>BB</em></strong> = <strong role="strong"><em>p</em></strong> X <strong role="strong"><em>p</em></strong> = <strong role="strong">p</strong><sup>2</sup> and</p><p>Probability of genotype <strong role="strong"><em>Bb</em></strong>= (<strong role="strong"><em>p</em></strong> X <strong role="strong"><em>q</em></strong>) + (<strong role="strong"><em>q</em></strong> X <strong role="strong"><em>p</em></strong>) = <strong role="strong">2 pq</strong> and</p><p>Probability of genotype <strong role="strong"><em>bb</em></strong> = <strong role="strong"><em>q</em></strong> X <strong role="strong"><em>q</em></strong> = <strong role="strong">q</strong><sup>2</sup></p><p>We have included all possible genotypes, so the probabilities must add to 1.0. Our equation becomes:</p><div><p><strong>Table </strong>1.2 </p><table><p></p><thead><tr><td><strong role="strong"><em>p</em></strong><strong role="strong"><sup>2</sup></strong></td><td>+</td><td><strong role="strong">2</strong> <strong role="strong"><em>pq</em></strong></td><td>+</td><td><strong role="strong"><em>q</em></strong><strong role="strong"><sup>2</sup></strong></td><td>=</td><td><strong role="strong">1</strong></td></tr></thead><tbody><tr><td>frequency of genotype <em>BB</em></td><td> </td><td>frequency of genotype <em>Bb</em></td><td> </td><td>frequency of genotype <em>bb</em></td><td> </td></tr></tbody></table></div><p>This is the Hardy-Weinberg equation, which describes the relationship between allele frequencies and genotype frequencies for a population at equilibrium.</p><p>The equation can be used to determine genotype frequencies if allele frequencies are known, or allele frequencies if genotype frequencies are known. Let&rsquo;s use a common human genetic disease as an example. Cystic fibrosis (CF) is caused by a recessive allele (<strong role="strong"><em>f</em></strong>) which makes a non-functional chloride ion channel, leading to excessive mucus in the lungs, inadequate enzyme secretion by the pancreas, and early death (<strong role="strong">Figure</strong>  1.12 <a href="#Bio13-2-2-fig" id="Bio13-2-2-text">below</a>).</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2832" width="256px" /></div> <p><a href="#Bio13-2-2-text" id="Bio13-2-2-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.12 <caption>Cystic fibrosis is an inherited disease of the lungs and pancreas. A recessive allele of a gene for a chloride ion channel  located on chromosome 7 (the gene locus is colored red, right) causes the disease. Treatment includes ventilator and antibiotic therapy (above).</caption></p></div><p> </p><p>Knowing that 1 of every 3,900 children in the United States is born with CF, we can use the Hardy-Weinberg equation to ask what proportion of the population unknowingly carries the allele for cystic fibrosis. An individual who has CF must have the genotype <strong role="strong"><em>ff</em></strong>, because the allele is recessive. Using the value for the frequency of the homozygous recessive genotype, we can calculate q, the frequency of the recessive allele,<strong role="strong"><em>f:</em></strong></p><p>1/3900 = 0.0002564 = frequency of <strong role="strong"><em>ff</em></strong> genotypes = <strong role="strong"><em>q</em></strong><strong role="strong"><sup>2</sup></strong></p><p>To find <strong role="strong"><em>q</em></strong>, the frequency of allele <strong role="strong"><em>f</em></strong>, we must take the square root of the frequency of <strong role="strong"><em>ff</em></strong> genotypes:</p><p><strong role="strong"><em>q</em></strong> = &radic;0.0002564 = 0.016 = frequency of the <strong role="strong"><em>f</em></strong> allele</p><p>Because <strong role="strong"><em>p</em></strong> <strong role="strong">+</strong> <strong role="strong"><em>q</em></strong> <strong role="strong">= 1</strong> (the sum of the frequency of allele <strong role="strong"><em>f</em></strong> and the frequency of allele <strong role="strong"><em>F</em></strong> must equal 1.0),</p><p><strong role="strong"><em>p</em></strong> <strong role="strong">= 1 &ndash;</strong> <strong role="strong"><em>q</em></strong> = 1.0 &ndash; 0.0160 = 0.9840 = frequency of the <strong role="strong"><em>F</em></strong> allele</p><p>According to the Hardy-Weinberg equation, for a population at equilibrium the frequency of carrier genotypes, <strong role="strong"><em>Ff</em></strong>, is <strong role="strong">2pq</strong> = 2 X 0.016 X 0.984 = <strong role="strong">0.0315</strong>.</p><p>In other words, if the population is indeed at equilibrium for this gene, just over 3% of the population carries the recessive allele for cystic fibrosis.</p><p>Of course, this calculation holds only true if the US population meets the five conditions we listed at the beginning of this section. In nature, populations seldom satisfy all five criteria. Let&rsquo;s consider how well each condition describes the US population for the cystic fibrosis gene:</p><p><strong role="strong">Very Large Population Size</strong></p><p>Although the equation ideally describes an infinitely large population &ndash; never found in nature, of course &ndash; the US population is probably large enough that this condition alone does not significantly disrupt equilibrium.</p><p><strong role="strong">No Mutation</strong></p><p>Mutations happen constantly, if at a low rate, so &ldquo;no mutation&rdquo; is a second unrealistic condition. However, mutations affecting any one particular gene are rare, so their effect on an otherwise large, stable population is small.</p><p><strong role="strong">No Migration</strong></p><p>This condition assumes no net additions or losses of either allele to the gene pool through immigration or emigration. For the US population, immigration is probably more significant than emigration. <strong role="strong">Gene flow,</strong> in essence is the flow of alleles into or out of a population, may be the most significant problem for this particular gene, because the frequency of the allele for cystic fibrosis varies greatly according to ancestry. Although 1 in 25 Europeans carry the <strong role="strong"><em>f</em></strong> allele, the frequency is just 1 in 46 among Hispanics, 1 in 60 among Africans, and 1 in 90 among Asians. Therefore, disproportionate immigration by certain groups changes allele frequencies, destabilizing the Hardy-Weinberg equilibrium.</p><p><strong role="strong">Random Mating</strong></p><p>Here is another assumption which is probably not realistic. Marriage between individuals of similar ancestry and culture is still more common than intermarriage, although both occur. If marriages are not random, Hardy-Weinberg equilibrium does not apply.</p><p><strong role="strong">No Natural Selection</strong></p><p>The final condition is that all genotypes must have an equal chance to survive and reproduce. Victims of cystic fibrosis (genotype <strong role="strong"><em>ff</em></strong>) have shorter lifespans, which inevitably reduce reproduction compared to individuals without the disease (genotypes <strong role="strong"><em>FF</em></strong> and <strong role="strong"><em>Ff</em></strong>). Although medical care is improving, differential survival and reproduction among genotypes means the gene pool is not at equilibrium.</p><p>With respect to the cystic fibrosis gene, the U.S. population fails to meet at least three of the criteria for equilibrium. Therefore, the actual frequencies of alleles and genotypes probably deviate somewhat from those we calculated.</p><p>In nature, very few populations meet the Hardy-Weinberg requirements for equilibrium. If we look at this fact from a different angle, we see that any of these conditions can destabilize equilibrium, causing a change in allele frequencies. In other words, these five conditions are five major causes of microevolution. The remaining sections of this lesson will explore each cause in detail.</p><h3>Causes of Microevolution: Mutation and Gene Flow</h3><p>As discussed in the last lesson, mutation &ndash; a random, accidental change in the sequence of nucleotides in DNA &ndash; is the original source of genetic variation. Only mutation can create new alleles &ndash; new raw material for natural selection. UV or ionizing radiation, chemicals, and viruses constantly generate mutations in a gene pool, destabilizing genetic equilibrium and creating the potential for adaptation to changing environments. However, both rates of mutation and their effects on the fitness of the organism vary.</p><p>In multicellular organisms, most mutations occur in body cells and do not affect eggs and sperm; these are lost when the individual dies and usually do not affect evolution. Only mutations in gamete-producing cells can become part of the gene pool. The rate at which mutations enter the gene pool is low, due to DNA &ldquo;proofreading&rdquo; and repair enzymes - and the extensive amount of &ldquo;junk&rdquo; DNA which does not code for protein. Mutations which do change nucleotide sequences in functional genes may also have no effect, because the Genetic Code is redundant (multiple similar codons for a single amino acid), or very little effect, if the amino acid is not located in a critical part of the protein.</p><p>Occasionally, however, a single nucleotide substitution can have a major effect on a protein &ndash; as we saw with sickle-cell anemia in the last lesson. Usually, the effect of a mutation on a protein is harmful; rarely is it helpful. In sickle-cell anemia, it is both &ndash; in certain environments. Sickled cells carry oxygen much less efficiently, but prevent malaria infections. Overall, the chance that a single mutation will increase the fitness of a multicellular organism is extremely low. If the environment changes, however, the adaptive value of a new allele may change as well. Over time, mutations accumulate, providing the variation needed for natural selection.</p><p>For the small genomes of viruses and bacteria, mutations affect genes directly and generation times are short, so rates of mutation are much higher. For an HIV population in one AIDS patient, rates of viral mutation and replication are so high that in a single day, every site in the HIV genome may have experienced mutation (<strong role="strong">Figure</strong>  1.13 <a href="#Bio132-03-fig" id="Bio132-03-text">below</a>). This rapid generation of new alleles challenges our best efforts at drug treatment, and explains the evolution of drug antibiotic resistance. Because of the abundance of random, spontaneous mutations, HIV generates a large amount of raw material for natural selection, and readily evolves resistance to new &ldquo;environments&rdquo; created by single drugs (<strong role="strong">Figure</strong>  1.14 <a href="#Bio132-04-fig" id="Bio132-04-text">below</a>). Drug &ldquo;cocktails,&rdquo; which contain multiple anti-viral chemicals, are our effort to change the &ldquo;environment&rdquo; and keep up with mutation in the human-HIV evolutionary race. For microorganisms, mutation is a strong force for evolution.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2813" width="256px" /></div> <p><a href="#Bio132-03-text" id="Bio132-03-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.13 <caption>HIV daughter particles are shed from an infected human T-cell host. HIV replication and mutation rates are so high that during a single day, the HIV population in one AIDS patient generates mutations at every site in the HIV genome. New alleles provide the potential for extremely rapid evolution, including the development of resistance to drugs.</caption></p></div><p> </p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2801" width="256px" /></div> <p><a href="#Bio132-04-text" id="Bio132-04-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.14 <caption>Extremely high HIV mutation rates provide many new alleles  raw material for natural selection  in AIDS patients. This variation allows at least some mutant viruses to survive and reproduce in the changing environments of new drug therapies. The graph shows the effects of one type of drug: an initial decrease in the HIV population and a temporary rise in the number of human host (CD4) cells. Before long, however, mutants have alleles conferring drug resistance appear and begin to reproduce, and the population recovers. This change in allele frequencies is microevolution, caused by mutation and natural selection.</caption></p></div><p> </p><p>For all organisms, mutations are the ultimate source of genetic variation. For a population, however, the immigration or emigration of individuals or gametes may also add to or subtract from a gene pool &ndash; a process known as <strong role="strong">gene flow</strong>. For example, wind or animals can carry pollen or seeds from one plant population to another. In baboon troops or wolf packs (<strong role="strong">Figure</strong>  1.15 <a href="#Bio132-05-fig" id="Bio132-05-text">below</a>), juvenile males may leave the group to find mates and establish separate populations. Human history includes countless migrations; gene flow continues to mix gene pools and cause microevolutionary change.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2828" width="256px" /></div> <p><a href="#Bio132-05-text" id="Bio132-05-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.15 <caption>Dominant alpha wolves lead their pack in Yellowstone National Park. The lowest ranking omega individual, in the rear, may eventually leave the pack to find a mate and establish his own territory and pack. He would carry some of the genes from his pack to the new one  a form of gene flow which seems built in to wolves social organization.</caption></p></div><p> </p><p>Gene flow can bring into a population new alleles which occurred by chance and were successful in other populations. In this way, it can accelerate microevolution. However, if exchange between populations is frequent, it reduces differences between populations, in effect increasing population size. In this case, gene flow tends to maintain separate populations as one species, reducing speciation, if not microevolution.</p><p>Mutation, together with recombination of existing alleles by sexual reproduction (see lesson 13.1) provides the diversity which is the raw material for natural selection and evolution. Gene flow can accelerate the spread of alleles or reduce the differences between populations. Both can contribute significantly to microevolution. However, many biologists consider the major causes of microevolutionary change to be genetic drift and natural selection.</p><h3>Causes of Microevolution: Population Size and Genetic Drift</h3><p>Recall that the third requirement for Hardy-Weinberg equilibrium is a very large population size. This is because chance variations in allele frequencies are minimal in large populations. In small populations, random variations in allele frequencies can significantly influence the &ldquo;survival&rdquo; of any allele, regardless of its adaptive value. Random changes in allele frequencies in small populations are known as <strong role="strong">genetic drift</strong>. Many biologists think that genetic drift is a major cause of microevolution.</p><p>You see the effects of chance when you flip a coin. If you flipped a penny 4 times, you would not be too surprised if it came up heads 4 times and tails not at all. If you tossed it 100 times, you would be very surprised if the results were 100 heads and no tails. The larger the &ldquo;population&rdquo; of coin tosses, the lower the effects of chance, and the closer the results should match the expected 50-50 ratio. The same is true for populations. If we imagine a rabbit population with a very small gene pool of just 2 <strong role="strong"><em>B</em></strong> alleles and 2 <strong role="strong"><em>b</em></strong> alleles, it is not difficult to understand that occasionally, chance alone would result in no albino offspring (only genotypes <strong role="strong"><em>BB</em></strong> or <strong role="strong"><em>Bb</em></strong>) &ndash; or even no brown offspring (only genotype <strong role="strong"><em>bb</em></strong>). However, a gene pool of 100 <strong role="strong"><em>B</em></strong> alleles and 100 <strong role="strong"><em>b</em></strong> alleles would be very unlikely to produce a generation of offspring entirely lacking one allele or the other, despite having identical initial allele frequencies of 0.5.</p><p>Because chance governs meiosis and fertilization, random variations can influence allele frequencies, especially for small populations. Note that these chance variations can increase the frequency of alleles which have no adaptive advantages or disadvantages &ndash; or decrease the frequency of alleles which do have adaptive value. Genetic drift can result in extinction of an allele or an entire population &ndash; or rapid evolution (<strong role="strong">Figure</strong>  1.16 <a href="#Bio13-2-6-fig" id="Bio13-2-6-text">below</a>). Two sets of circumstances can create small populations for which genetic drift can have major consequences: the <strong role="strong">bottleneck effect</strong> and the <strong role="strong">founder effect.</strong></p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2800" width="256px" /></div> <p><a href="#Bio13-2-6-text" id="Bio13-2-6-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.16 <caption>Computer models show that the effect of small population size on allele frequencies is a significant increase in variation due to chance. Each line depicts a different allele. In the small population (above), most of 20 alleles, beginning at frequencies of 0.5, become either fixed (frequency = 1.0) or extinct (frequency = 0) within 5  25 generations. In the larger population (below), only one pair of alleles shows fixation/extinction  and that occurs only after 45 generations. Note that these variations are independent of natural selection; they do not necessary fit the organism to its environment.</caption></p></div><p> </p><h4>The Bottleneck Effect</h4><p>Natural catastrophes such as earthquakes, floods, fires, or droughts can drastically reduce population size &ndash; usually without respect to allele frequencies. As a result of the disaster, some alleles may be lost entirely, and others may be present in frequencies which differ from those of the original population. The smaller population is then subject to genetic drift, which may further reduce diversity within the population. The loss of diversity resulting from a drastic reduction in population size and subsequent genetic drift is the <strong role="strong">bottleneck effect</strong> (<strong role="strong">Figure</strong>  1.17 <a href="#Bio132-07-fig" id="Bio132-07-text">below</a>). Much of our concern for endangered species derives from our understanding of the way in which small population size can reduce diversity by increasing genetic drift. We will look at two examples of the bottleneck effect &ndash; one caused by humans, and the other probably experienced by our human ancestors.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2823" width="256px" /></div> <p><a href="#Bio132-07-text" id="Bio132-07-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.17 <caption>A random but major catastrophe which causes a sudden, severe reduction in population can lead to a bottleneck effect. The reduction in gene pool size - and often diversity - leaves populations subject to genetic drift; chance variations can cause extinction  or accelerated evolution leading to recovery. Even if recovery occurs, genetic diversity remains low.</caption></p></div><p> </p><p>During the 19<sup>th</sup> century, overhunting reduced the worldwide population of Northern Elephant Seals (<strong role="strong">Figure</strong>  1.18 <a href="#Bio132-08-fig" id="Bio132-08-text">below</a>) to fewer than 100 individuals. Because an alpha bull typically mates with a &ldquo;harem&rdquo; of 30-100 females, it is possible that just a single male fathered all the seals which exist today! After legal protection, their numbers have rebounded to 100,000. However, the effects of the past bottleneck - significant loss of genetic variability &ndash; remain. Reduced genetic diversity means that today&rsquo;s seals are more susceptible to disease and weather. Effects of genetic drift on the gene pool may have contributed to the loss of 80% of pups during the El Nino year of 1997-98.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2831" width="256px" /></div> <p><a href="#Bio132-08-text" id="Bio132-08-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.18 <caption>The Northern Elephant Seal population fell to fewer than 100 individuals due to overhunting during the 19 century. Although their numbers have recovered, the bottleneck effect of reduced genetic variation limits their potential to adapt to future environmental changes.</caption></p></div><p> </p><p>Although the exact cause is unknown, a bottleneck for South African Cheetahs (<strong role="strong">Figure</strong>  1.19 <a href="#Bio132-09-fig" id="Bio132-09-text">below</a>) during the last ice age about 10,000 years ago has apparently led to extremely low genetic variability. Genetic variation among cheetahs has been compared to that of highly inbred varieties of laboratory mice; skin grafts between unrelated individuals are not rejected. These animals also suffer from low sperm counts. Like many endangered species, cheetahs are threatened not only by habitat loss, but also by reduced genetic diversity, which reduces their potential to adapt to changing environments.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2820" width="256px" /></div> <p><a href="#Bio132-09-text" id="Bio132-09-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.19 <caption>Although the cause is unknown, South African Cheetahs apparently experienced a population bottleneck 10,000 years ago. Their current genetic uniformity is remarkable; skin grafts between even unrelated individuals do not elicit rejection responses.</caption></p></div><p> </p><p>We humans may have experienced a population bottleneck between 70,000 and 75,000 years ago, when supervolcano Mount Toba exploded with category 8 (&ldquo;megacolossal!&rdquo;) force in Sumatra. According to anthropologist Stanley Ambrose&rsquo;s theory, global temperature dropped as much as 5 degrees Celsius for several years, possibly leading to an ice age. Ambrose believes that the environmental effects (&ldquo;six years of relentless volcanic winter&rdquo;) reduced the total human population to less than 10,000, and that isolated individual populations would have experienced genetic drift and rapid evolution or extinction.</p><h4>The Founder Effect</h4><p>Whereas a drastic reduction in population size causes the bottleneck effect, a form of population expansion leads to the <strong role="strong">founder effect.</strong> If a small group of individuals (the founders) breaks off from a larger population to colonize a distant area, they will probably carry with them only a limited amount of the genetic diversity of the original population (<strong role="strong">Figure</strong>  1.20 <a href="#Bio13-02-10-fig" id="Bio13-02-10-text">below</a>). For this reason, the new population they establish may differ significantly in genotype and phenotype. Inevitably, it will also be small and therefore subject to genetic drift.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2799" width="256px" /></div> <p><a href="#Bio13-02-10-text" id="Bio13-02-10-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.20 <caption>In this diagram, the red squares and blue dots represent individuals with different alleles. Small groups of individuals (see the right three circles) which leave a larger population (left) to colonize a new area carry with them smaller gene pools with allele frequencies which may differ significantly from that of the parent population  due to chance. The effects of chance in these new populations become more important; genetic drift may result in extinction or rapid evolution.</caption></p></div><p> </p><p>On newly formed islands, such as the Galapagos, Hawaii, and more recently Surtsey, Iceland, founder populations are often the only source of life on the island. Many founder populations probably become extinct, but others evolve rapidly, due to genetic drift. Some may diverge rapidly to occupy many available ecological niches &ndash; a process known as <strong role="strong">adaptive radiation.</strong> As we have discussed in past chapters, Galapagos finches and Hawaiian honeycreepers probably each evolved from small populations of a single ancestral finch-like species (see the previous chapter on <em>Evolutionary Theory</em>).</p><p>Historically and even today, human populations have experienced founder effects. In some cases, migration and colonization are the cause. Quebec was founded by a group of no more than 2,600 people, ancestors of today&rsquo;s more than 7 million Quebecois, who show remarkable genetic similarity and a number of heritable diseases, well studied by geneticists.</p><p>Cultural isolation, as well as colonization, can result in founder effects. Amish populations in the United States have grown from an initial group of about 200 immigrants, dating back to the mid-1700s. Because they have remained culturally and reproductively isolated from non-Amish Americans, they show considerable uniformity. The Amish today are often studied for their genetic uniformity, as well as certain recessive conditions. Geneticists believe that just one or two of the initial 200 Amish carried a recessive allele for Ellis-van Creveld syndrome (short limbs, extra fingers, and heart anomalies), yet through genetic drift, the isolated Amish population now has the highest incidence of this syndrome in the world (<strong role="strong">Figure</strong>  1.21 <a href="#Bio132-11-fig" id="Bio132-11-text">below</a>).</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2830" width="256px" /></div> <p><a href="#Bio132-11-text" id="Bio132-11-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.21 <caption>Polydactyly (extra digits) and short limbs are characteristics of Ellis-van Creveld syndrome, a genetic disease which is rare worldwide but more common in the Amish population. Because the Amish population began with just 200 immigrants and has remained isolated, their high incidence of this syndrome may be a result of the founder effect.</caption></p></div><p> </p><h3>Natural Selection</h3><p>While genetic drift, including the bottleneck and founder effects, can cause microevolution (generational change in allele frequencies), its effects are mostly random. The results of genetic drift may include enhanced capabilities, but more often, they are neutral or deleterious. Natural selection depends not on chance, but on differential survival determined by an individual&rsquo;s traits. Even though the variations are due to chance, the products of natural selection are usually organisms well-suited to their environment.</p><p>Recall that Hardy-Weinberg equilibrium requires that all individuals in a population are equal in their ability to survive and successfully reproduce. As Darwin noted, however, overproduction of offspring and variation among individuals often lead to differential survival and reproduction &ndash; in other words, natural selection (<strong role="strong">Figure</strong>  1.22 <a href="#Bio13-02-12-fig" id="Bio13-02-12-text">below</a>). We discussed natural selection as a part of Darwin&rsquo;s theory of evolution in the last chapter, but in this section, we will go deeper than Darwin could. We will explore natural selection at the level of populations &ndash; in terms of allele, genotype, and phenotype frequencies.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2822" width="256px" /></div> <p><a href="#Bio13-02-12-text" id="Bio13-02-12-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.22 <caption>Natural selection involves (1) heritable variation (here, giraffe neck length); (2) overproduction of offspring (3 giraffes born, not all can survive); (3) differential survival and reproduction (not enough food for all giraffes; those with shorter necks starve); and (4) gradual change in traits in the population (long-necked giraffes survive and reproduce, so their genes for long necks increase in frequency in the next generation).</caption></p></div><p> </p><h4>Acting on an Organism's Phenotype</h4><p>Natural selection acts on an organism&rsquo;s phenotype (appearance), which is a product of genotype and any environmental influences on gene expression. By selecting <em>for</em> alleles which improve survival and/or reproduction and selecting <em>against</em> harmful alleles, natural selection changes the proportion of alleles from one generation to the next &ndash; causing microevolution. Let&rsquo;s return once more to our rabbit population. If a predator such as a hawk can see white rabbits (genotype <strong role="strong"><em>bb</em></strong>) more easily than brown rabbits (<strong role="strong"><em>BB</em></strong> and <strong role="strong"><em>Bb</em></strong>), brown rabbits are more likely than white rabbits to survive hawk predation. Because more brown rabbits will survive to reproduce, the next generation will probably contain a higher frequency of <strong role="strong"><em>B</em></strong> alleles. Note, however, that the recessive <strong role="strong"><em>b</em></strong> alleles are unlikely to disappear completely, because they can &ldquo;hide&rdquo; from the hawks in heterozygous brown rabbits. This is a good reminder that natural selection acts on phenotypes, rather than genotypes. The hawk - or natural selection - is unable to distinguish a <strong role="strong"><em>BB</em></strong> rabbit from a <strong role="strong"><em>Bb</em></strong> rabbit. Natural selection - and the hawk - is only able to distinguish a brown rabbit from a white rabbit, demonstrating how natural selection acts on the phenotype rather than the genotype of an organism.</p><p>Consider a different example, which emphasizes reproduction rather than survival: If both brown and white rabbits preferred to mate with white rabbits, the next generation&rsquo;s gene pool would probably show an increase in the frequency of the <strong role="strong"><em>b</em></strong> allele, because white rabbits would be more likely to reproduce successfully.</p><p>Although some traits are determined by a single gene, many are influenced by more than one gene (polygenic). The result of polygenic inheritance is a continuum of phenotypic values which often show a bell curve pattern of variation. <strong role="strong">Figure</strong>  1.23 <a href="#Bio132-15-fig" id="Bio132-15-text">below</a> shows the effect of three genes, each having two alleles, on human skin color; the result is a normal distribution ranging from very dark to very light, with a peak near the middle. You can demonstrate polygenic inheritance (probably with some environmental influence) for height, ear length, or handspan by measuring your classmates and graphing the data in a similar fashion. Some curves will be flat, and others sharp &ndash; but most will resemble the normal &ldquo;bell&rdquo; shape.</p><p>Given this pattern of phenotypic variation, natural selection can take three forms (<strong role="strong">Figure</strong>  1.24 <a href="#Bio1302-14-fig" id="Bio1302-14-text">below</a>). We will use the theoretical human skin color distribution <strong role="strong">Figure</strong>  1.23 <a href="#Bio132-15-fig" id="Bio132-15-text">below</a> to illustrate the three types of selection. <strong role="strong">Directional selection</strong> shifts the frequency curve away from the average by favoring individuals with an extreme form of the variation. The skin of early humans living in sun-rich Africa received high levels of UV radiation, which destroys vitamin B (folate) and leads to severe birth defects such as spina bifida. Selection, then, favored darker-skinned individuals, and the frequency of the darker alleles increased. After several generations, the curve would still be bell-shaped, but it would have shifted to the right, in the direction of the darker alleles. The average individual would have had darker skin as result of this microevolutionary change.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2824" width="256px" /></div> <p><a href="#Bio1302-14-text" id="Bio1302-14-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.24 <caption>Three types of selection can alter allele frequencies, causing microevolution. The effect of stabilizing selection (1) is to reduce variation. Disruptive selection (2) results in two different populations, which may eventually be isolated from one another. Directional selection (3) enhances or reduces a single characteristic, such as trunk or snout length in the above example.</caption></p></div><p> </p><h4>Natural Selection and Human Migration</h4><p>As humans migrated into the northern hemisphere, excessive UV radiation was no longer a problem, but the relative lack of sunlight led to lower levels of vitamin D<sub>3</sub>, normally synthesized in the skin and necessary for calcium absorption and bone growth. Thus, selection in the north favored lighter-skinned individuals &ndash; by itself another example of directional selection. However, if we consider the human population as a whole at that time, <strong role="strong">disruptive selection</strong> would describe the microevolution taking place. In northern climates, alleles for light skin would be favored, and in southern climates would select for alleles for dark skin, resulting in two distinct peaks in the distribution of skin color phenotypes and their corresponding genotypes. Keep in mind that the &ldquo;three gene &ndash; dark/light&rdquo; model is an oversimplification of the genetics underlying skin color, but the adaptive values are real, and the model allows us to illustrate how microevolution works. Note that a map of human skin colors supports this type of selection to some extent (<strong role="strong">Figure</strong>  1.23 <a href="#Bio132-15-fig" id="Bio132-15-text">below</a>).</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2802" width="256px" /></div> <p><a href="#Bio132-15-text" id="Bio132-15-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.23 <caption>The distribution of skin colors at least partially supports disruptive selection  for lighter skin in the north, to allow sunlight to form vitamin D in the skin, and for darker skin toward the equator to prevent UV radiation from breaking down vitamin B-folate.</caption></p></div><p> </p><p>Today, extensive migration, mobility, and intermarriage in the human population may be changing selective pressures on skin color once again. For the sake of argument, let&rsquo;s make the somewhat unrealistic assumption that mixing becomes complete and that all people will be sufficiently mobile that they experience intermediate levels of sunlight. These conditions would select against both extremely dark skin (too little vitamin D<sub>3</sub>) and extremely light skin (too little vitamin B-folate). The result would be a taller, narrower distribution &ndash; less diversity - about the same mean, a phenomenon known as <strong role="strong">stabilizing selection</strong>. Although our example is perhaps unrealistic, stabilizing selection is probably the most common form of natural selection, preventing form and function from straying away from a &ldquo;proven&rdquo; norm.</p><h4>Stabilizing Selection</h4><p>Stabilizing selection can lead to the preservation of harmful alleles. A famous example, which we considered in earlier lessons, is sickle-cell anemia. The gene for Beta-hemoglobin - half of the oxygen-carrying protein in our blood - has two alleles, which we will call Hgb-A and Hgb-S. Individuals having two copies of the Hgb-S allele suffer from sickle-cell anemia, a potentially lethal disease in which sickled cells clog capillaries and cannot carry oxygen efficiently. In equatorial regions, individuals with two copies of Hgb-A become infected with <em>Plasmodium</em> parasites and often die from malaria. However, individuals with one copy of each allele (the heterozygous genotype) escape both causes of death; although they may experience slight sickling at high altitudes, they do not suffer from full-blown anemia, and malaria parasites cannot infect their red blood cells. Stabilizing selection has maintained the frequencies of both alleles, even though each is potentially lethal in the homozygous state.</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2825" width="256px" /></div> <p><a href="#Bio13-2-16-text" id="Bio13-2-16-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.25 <caption>The distribution of malaria (top) correlates closely with the distribution of the sickle-cell allele (bottom). Because the heterozygous genotype confers immunity to malaria, this allele which is lethal in the homozygous condition persists in environments where malaria is common. Thus, natural selection can occasionally result in persistence of harmful alleles.</caption></p></div><p> </p><p>Selection for a particular trait may also select for other traits which do not directly affect fitness &ndash; if, for example, genes are linked, or if a single gene influences several different traits.</p><h4>Fitness</h4><p>Another way to look at natural selection is in terms of <strong role="strong">fitness</strong> - the ability of an organism with a certain genotype to reproduce. Fitness can be measured as the proportion of that organism&rsquo;s genes in all of the next generation&rsquo;s genes. When differences in individual genotypes affect fitness, the genotypes with higher fitness become more common. This change in genotype frequencies is natural selection.</p><h4>Kin Selection</h4><p>An intriguing corollary of genotype selection is <strong role="strong">kin selection.</strong> Behaviors which sacrifice reproductive success or even survival can actually increase fitness if they promote the survival and reproduction of close relatives who share a significant proportion of the same genes. Examples include subordinate male turkeys, who help their dominant brothers display to potential mates (<strong role="strong">Figure</strong>  1.26 <a href="#Bio132-17-fig" id="Bio132-17-text">below</a>) and honeybee workers, who spend their lives collecting pollen and raising young to ensure that their mother, the queen, reproduces successfully (<strong role="strong">Figure</strong>  1.27 <a href="#Bio132-18-fig" id="Bio132-18-text">below</a>).</p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2807" width="256px" /></div> <p><a href="#Bio132-17-text" id="Bio132-17-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.26 <caption>Wild turkeys display in groups of closely related individuals, but only alpha males eventually mate. Subordinate males sacrifice their chance to reproduce, even chasing away other males to promote their dominant brothers success, because this behavior increases the chance that the genes they share will be represented in the next generation. This means of increasing gene frequency is kin selection.</caption></p></div><p> </p><p> </p><div style="float: left; clear: both; padding: 3em; width: 256px;" class="figure"><div><img src="http://pioneer.ck12.org/flexr/api/ext/images?id=2803" width="256px" /></div> <p><a href="#Bio132-18-text" id="Bio132-18-fig">[Back to Text]</a></p><p><strong>Figure </strong>1.27 <caption>Many social insects also illustrate kin selection. These honeybee workers are sterile. They spend their lives collecting pollen, feeding larvae, and cleaning and defending the hive. With no chance of reproductive success of their own, they dedicate their lives to the reproductive success of the hives queen  their mother  who shares 50% of her genes with each of them.</caption></p></div><p> </p><p>We have looked carefully at equilibrium populations and at possible disruptions of equilibrium which cause microevolution &ndash; a generational change in a population&rsquo;s allele frequencies:</p><ul><li>Mutation, which together with sexual reproduction is the ultimate source of variation, and is an important cause of microevolution in microorganisms </li><li>Gene flow, which can accelerate microevolution by importing new, already successful alleles </li><li>Genetic drift, which can increase the effect of chance variations in small populations </li><li>Natural selection, which can be directional, disruptive, or stabilizing </li><li>Specialized types of selection, such as mate selection and kin selection </li></ul><p>Evolutionary biologists are not yet in agreement regarding the relative importance of each type of selection to the history of life, although most would agree that natural selection is the primary force in microevolution. In the next lesson, we will apply our understanding of microevolutionary processes to that &ldquo;mystery of mysteries,&rdquo; as Darwin and Herschel called it: the origin of species.</p><h3>Lesson Summary</h3><ul><li>Macroevolution &ndash; change in species over geologic time &ndash; is the cumulative effect of microevolution. </li><li>Microevolution &ndash; evolution within species or populations &ndash; can be measured as a generation-to-generation change in allele frequencies. </li><li>Non-evolving populations have constant frequencies of alleles and genotypes &ndash; genetic equilibrium. </li><li>The Hardy-Weinberg model holds for equilibrium populations under 5 conditions: </li></ul><ol start><li>no mutation </li><li>no migration </li><li>very large population size </li><li>random mating </li><li>no natural selection </li></ol><ul><li>A generalized form of the Hardy-Weinberg equation for a gene pool of two alleles at equilibrium is: </li></ul><div><p><strong>Table </strong>1.2 </p><table><p></p><thead><tr><td><strong role="strong"><em>p</em></strong><strong role="strong"><sup>2</sup></strong></td><td>+</td><td><strong role="strong">2</strong> <strong role="strong"><em>pq</em></strong></td><td>+</td><td><strong role="strong"><em>q</em></strong><strong role="strong"><sup>2</sup></strong></td><td>=</td><td><strong role="strong">1</strong></td></tr></thead><tbody><tr><td>frequency of genotype BB</td><td> </td><td>frequency of genotype Bb</td><td> </td><td>frequency of genotype bb</td><td> </td></tr></tbody></table></div><p>where <strong role="strong"><em>p</em></strong> is the frequency of one allele and <strong role="strong"><em>q</em></strong> the frequency of the other allele.</p><ul><li>In nature very few populations meet the Hardy-Weinberg requirements for equilibrium, due to constant mutation, gene flow, small populations, nonrandom mating, and environmental change. </li><li>Random, spontaneous mutations constantly generate new alleles in a gene pool, destabilizing genetic equilibrium and creating the potential for adaptation to changing environments. </li><li>In multicellular organisms, only mutations that affect germ cells become part of the gene pool. </li><li>Because of extensive &ldquo;junk&rdquo; DNA, repair enzymes, and multicellularity, many mutations do not reach the gene pool. </li><li>Many mutations are harmful, disrupting protein function. </li><li>Because the genetic code is redundant and some amino-acid substitutions may not change protein function, many mutations have no effect on an organism&rsquo;s fitness. </li><li>Even neutral mutations hold potential for future selection if the environment changes. </li><li>A few mutations may be advantageous, improving or changing protein function. </li><li>In microorganisms mutation rates are much higher due to rapid reproduction and small genomes. </li><li>Mutation, together with recombination of existing alleles provides the diversity which is the raw material for natural selection and evolution. </li><li>The movement of genes from one population to another (gene flow) can change allele frequencies. </li><li>Gene flow can accelerate the spread of successful alleles or reduce differences between populations. </li><li>In small populations random variations in allele frequencies can significantly influence the &ldquo;survival&rdquo; of any allele, regardless of its adaptive value; this phenomenon is genetic drift. </li><li>Genetic drift can result in extinction of an allele or an entire population. </li><li>Alternatively, genetic drift can lead to rapid evolution of a population. </li><li>In the bottleneck effect, a catastrophe or disease or overhunting dramatically reduces a population&rsquo;s size and genetic variation, increasing its susceptibility to the effects of genetic drift. </li><li>In the founder effect, a small group leaves a larger population to colonize a new area. Again, genetic drift may lead to loss of genetic diversity, extinction, or rapid evolution. </li><li>Genetic drift leads to evolution in populations of small size, but results are mostly due to chance. </li><li>Natural selection occurs in populations of any size, and results are more likely to adapt a population to its environment. </li><li>Natural selection acts on phenotype variations, so may include environmental effects which are not heritable. </li><li>Evolution is a result of natural selection, and is measured as a change in allele frequencies. </li><li>For a trait whose genetic basis is polygenic, the pattern of phenotypic variation usually forms a bell curve about an average value. </li><li>Directional selection results in a shift of allele frequencies toward one extreme. </li><li>Disruptive selection favors both extremes over the average phenotypic value. </li><li>Stabilizing selection maintains or narrows existing variation in phenotype. </li><li><strong role="strong">Fitness</strong> is the ability of an organism with a certain genotype to reproduce successfully. </li><li>Because alleles often affect more than one trait in different ways, or have different effects in different environments, natural selection can sometimes lead to the persistence of harmful or even lethal allele. </li><li>Kin selection involves the sacrifice by an individual of his/her reproductive potential in order to help a close relative reproduce successfully. </li></ul><h3>Review Questions</h3><ol start><li>Define microevolution in terms of allele frequencies.</li><li>Describe genetic equilibrium, including the Hardy-Weinberg conditions.</li><li>In the United States, about 1 in every 12 black people of African descent carry one copy of the allele for sickle-cell anemia. Assuming the five conditions of Hardy-Weinberg equilibrium hold (we&rsquo;ll consider why they probably don&rsquo;t in question #4), calculate the probability that a member of the next generation will suffer from sickle-cell anemia (have the homozygous Hemoglobin S genotype).</li><li>Discuss the reasons why the 5 conditions for Hardy-Weinberg equilibrium are probably not met by the black population for the Hemoglobin alleles.</li><li>Analyze the possible effects of mutation, and the probability and importance of each.</li><li>Describe two possible effects of gene flow on the genetics of a population.</li><li>Describe three possible effects of genetic drift on populations and/or specific alleles.</li><li>Why do we consider Northern Elephant seals endangered, even though their population has risen to 100,000 individuals?</li><li>Use the distribution of phenotypes for a trait whose genetic basis is polygenic to interpret directional, disruptive, and stabilizing patterns of selection.</li><li>Describe how natural selection can sometimes lead to the persistence of harmful or even lethal allele. Include an example.</li></ol><h3>Further Reading / Supplemental Links</h3><ul><li>&ldquo;The Gene School:&rdquo; </li><li><a href="http://library.thinkquest.org/19037/population.html"> </a><a href="http://library.thinkquest.org/19037/population.html">http://library.thinkquest.org/19037/population.html</a></li><li><a href="http://genetics-education-partnership.mbt.washington.edu/class/activities/HS/sickle-bean.htm"> </a><a href="http://genetics-education-partnership.mbt.washington.edu/class/activities/HS/sickle-bean.htm">http://genetics-education-partnership.mbt.washington.edu/class/activities/HS/sickle-bean.htm</a></li><li><a href="http://www.koshland-science-museum.org/teachers/idactivity-act018.jsphttp://chroma.gs.washington.edu/outreach/genetics/sickle/index.html"> </a><a href="http://www.koshland-science-museum.org/teachers/idactivity-act018.jsp">http://www.koshland-science-museum.org/teachers/idactivity-act018.jsp</a></li><li><a href="http://chroma.gs.washington.edu/outreach/genetics/sickle/index.html"> </a><a href="http://chroma.gs.washington.edu/outreach/genetics/sickle/index.html">http://chroma.gs.washington.edu/outreach/genetics/sickle/index.html</a></li><li><a href="http://www.umsl.edu/~biology/hwec/MO-STEP/lessons/naturalsel.html"> </a><a href="http://www.umsl.edu/~biology/hwec/MO-STEP/lessons/naturalsel.html">http://www.umsl.edu/~biology/hwec/MO-STEP/lessons/naturalsel.html</a></li><li><a href="http://www.radford.edu/~rsheehy/Gen_flash/popgen/"> </a><a href="http://www.radford.edu/~rsheehy/Gen_flash/popgen/">http://www.radford.edu/~rsheehy/Gen_flash/popgen/</a></li><li><a href="http://www.phschool.com/science/biology_place/labbench/lab8/intro.html"> </a><a href="http://www.phschool.com/science/biology_place/labbench/lab8/intro.html">http://www.phschool.com/science/biology_place/labbench/lab8/intro.html</a></li><li><a href="http://sps.k12.ar.us/massengale/lab_8_sample2_ap_population_gene.htm"> </a><a href="http://sps.k12.ar.us/massengale/lab_8_sample2_ap_population_gene.htm">http://sps.k12.ar.us/massengale/lab_8_sample2_ap_population_gene.htm</a></li><li><a href="http://www.bgsu.edu/departments/chem/faculty/leontis/chem447/PDF_files/Jablonski_skin_color_2000.pdf"> </a><a href="http://www.bgsu.edu/departments/chem/faculty/leontis/chem447/PDF_files/Jablonski_skin_color_2000.pdf">http://www.bgsu.edu/departments/chem/faculty/leontis/chem447/PDF_files/Jablonski_skin_color_2000.pdf</a></li><li><a href="https://www3.nationalgeographic.com/genographic/"> </a><a href="https://www3.nationalgeographic.com/genographic/">https://www3.nationalgeographic.com/genographic/</a></li><li><a href="http://darwin.eeb.uconn.edu/simulations/simulations.html"> </a><a href="http://darwin.eeb.uconn.edu/simulations/simulations.html">http://darwin.eeb.uconn.edu/simulations/simulations.html</a></li><li><a href="http://www.berkeley.edu/news/media/releases/2005/03/02_turkeys.shtml"> </a><a href="http://www.berkeley.edu/news/media/releases/2005/03/02_turkeys.shtml">http://www.berkeley.edu/news/media/releases/2005/03/02_turkeys.shtml</a></li><li><a href="http://essp.csumb.edu/eseal/kristi_west/history.html"> </a><a href="http://essp.csumb.edu/eseal/kristi_west/history.html">http://essp.csumb.edu/eseal/kristi_west/history.html</a></li></ul><h3>Vocabulary</h3><dl><dt>adaptive radiation </dt><dd>Relatively rapid evolution of several species from a single founder population to several to fill a diversity of available ecological niches. </dd></dl><dl><dt>allele frequency </dt><dd>The fraction (usually expressed as a decimal) of a population&rsquo;s gene pool made up of a particular allele. </dd></dl><dl><dt>bottleneck effect </dt><dd>The loss of diversity resulting from a drastic reduction in population size and subsequent genetic drift. </dd></dl><dl><dt>directional selection </dt><dd>Selection which favors one side of a phenotypic distribution &ndash; one allele or one extreme of a normal distribution. </dd></dl><dl><dt>disruptive selection </dt><dd>Selection which favors the two extremes of a phenotypic distribution &ndash; the ends of a bell curve, or the homozygous phenotypes, as opposed to the average, or heterozygous phenotype. </dd></dl><dl><dt>fitness </dt><dd>The ability of an organism with a certain genotype to survive and reproduce, often measured as the proportion of that organism&rsquo;s genes in all of the next generation&rsquo;s genes. </dd></dl><dl><dt>founder effect </dt><dd>The loss of genetic diversity resulting from colonization of a new area by a small group of individuals which have broken off from a larger population. </dd></dl><dl><dt>gene flow </dt><dd>The net movement of genes into or out of a population through immigration or emigration. </dd></dl><dl><dt>genetic drift </dt><dd>Random changes in allele frequencies in small populations. </dd></dl><dl><dt>genetic equilibrium </dt><dd>State of a population in which allele and genotype frequencies remain constant from one generation to the next &ndash; a non-evolving population. </dd></dl><dl><dt>Hardy-Weinberg model </dt><dd>Describes a population at genetic equilibrium, meeting five conditions:  no mutation, no migration, very large population size, random mating, and no natural selection. </dd></dl><dl><dt>kin selection </dt><dd>Behaviors which sacrifice reproductive success or even survival to promote the survival and reproduction of close relatives who share a significant proportion of the same genes. </dd></dl><dl><dt>macroevolution </dt><dd>Evolution at or above the species level. </dd></dl><dl><dt>microevolution </dt><dd>Evolution within a species or population, also defined as a generation-to-generation change in allele frequencies for a population. </dd></dl><dl><dt>mutation </dt><dd>A change in the nucleotide sequence of DNA or RNA. </dd></dl><dl><dt>natural selection </dt><dd>The process by which a certain trait becomes more common within a population, including heritable variation, overproduction of offspring, and differential survival and reproduction. </dd></dl><dl><dt>polygenic trait </dt><dd>Traits that are influenced by more than one gene. </dd></dl><dl><dt>stabilizing selection </dt><dd>Selection which favors the average or heterozygous phenotype, resulting in no change or in a narrowing of the distribution of phenotypes. </dd></dl><h3>Points to Consider</h3><ul><li>This lesson discussed probable past microevolution of alleles for genes for human skin color and hemoglobin. For what other genes (or heritable traits) can you suggest past selective pressures? Do you think certain human genes (or heritable traits) may be at genetic equilibrium? Give some examples, and explain your reasoning. </li><li>Some people suggest that we humans have removed ourselves from natural selection. Do you agree? </li><li>What are some consequences of understanding that chance variations and natural selection can result in the persistence of lethal alleles, such as the alleles for sickle-cell anemia and cystic fibrosis? </li><li>Do you think it is important that people understand the biological basis of skin color? Explain </li></ul>